aerographene: ícones do tema pixelized (tabela pixelized/icons.html).

Origem: aerographene (original). Licença: GPL-3.0.
Depende de: aerographene/theme-icons-default.patch.
Propósito: 32 .icon de retângulos 16x16 (DISABLE_AA, FILL_RULE_NONZERO;
tons 50% e 22% por PATH_COLOR_ALPHA 0x80/0x38, tint mantido) gerados por
theming/design-language/gen_theme_icons.py e o mapa do tema pixelized.
A 20dip (escala 1,25) os pixels ficam com 1 ou 2 px de largura.

--- a/chrome/browser/ui/aerographene_theme_icons/BUILD.gn
+++ b/chrome/browser/ui/aerographene_theme_icons/BUILD.gn
@@ -43,6 +43,38 @@ aggregate_vector_icons("aerographene_theme_vector_icons") {
     "default_up.icon",
     "default_win.icon",
     # aerographene: theme-icons-pixelized
+    "pixelized_back.icon",
+    "pixelized_bell.icon",
+    "pixelized_check.icon",
+    "pixelized_chev.icon",
+    "pixelized_chevr.icon",
+    "pixelized_chl.icon",
+    "pixelized_chr.icon",
+    "pixelized_close.icon",
+    "pixelized_dl.icon",
+    "pixelized_down.icon",
+    "pixelized_ext.icon",
+    "pixelized_ext2.icon",
+    "pixelized_folder.icon",
+    "pixelized_fwd.icon",
+    "pixelized_gear.icon",
+    "pixelized_hist.icon",
+    "pixelized_info.icon",
+    "pixelized_link.icon",
+    "pixelized_max.icon",
+    "pixelized_min.icon",
+    "pixelized_minus.icon",
+    "pixelized_mon.icon",
+    "pixelized_more.icon",
+    "pixelized_plus.icon",
+    "pixelized_rel.icon",
+    "pixelized_search.icon",
+    "pixelized_side.icon",
+    "pixelized_spk.icon",
+    "pixelized_spkoff.icon",
+    "pixelized_split.icon",
+    "pixelized_up.icon",
+    "pixelized_win.icon",
     # aerographene: theme-icon-ublock
   ]
 }
--- a/chrome/browser/ui/aerographene_theme_icons/aerographene_theme_icons.cc
+++ b/chrome/browser/ui/aerographene_theme_icons/aerographene_theme_icons.cc
@@ -143,6 +143,122 @@ void AddDefaultIcons(gfx::VectorIconOverrideMap* map) {
 
 void AddPixelizedIcons(gfx::VectorIconOverrideMap* map) {
   // aerographene: theme-icons-pixelized
+  // i-back: views/toolbar/back_forward_button.cc:48-53
+  (*map)[&vector_icons::kArrowBackIcon] = &aerographene_theme_icons::kPixelizedBackIcon;
+  (*map)[&vector_icons::kBackArrowChromeRefreshOldIcon] = &aerographene_theme_icons::kPixelizedBackIcon;
+  (*map)[&::kBackArrowTouchOldIcon] = &aerographene_theme_icons::kPixelizedBackIcon;
+  // i-fwd: views/toolbar/back_forward_button.cc:62-67
+  (*map)[&vector_icons::kArrowForwardIcon] = &aerographene_theme_icons::kPixelizedFwdIcon;
+  (*map)[&vector_icons::kForwardArrowChromeRefreshOldIcon] = &aerographene_theme_icons::kPixelizedFwdIcon;
+  (*map)[&::kForwardArrowTouchOldIcon] = &aerographene_theme_icons::kPixelizedFwdIcon;
+  // i-rel: views/toolbar/reload_button.cc:68-73
+  (*map)[&vector_icons::kRefreshIcon] = &aerographene_theme_icons::kPixelizedRelIcon;
+  (*map)[&vector_icons::kReloadChromeRefreshOldIcon] = &aerographene_theme_icons::kPixelizedRelIcon;
+  (*map)[&::kReloadTouchOldIcon] = &aerographene_theme_icons::kPixelizedRelIcon;
+  // i-side: aerographene/sidebar-toggle-button.patch (sidebar_toggle_)
+  (*map)[&views::kMenuOpenIcon] = &aerographene_theme_icons::kPixelizedSideIcon;
+  (*map)[&views::kMenuOpenOldIcon] = &aerographene_theme_icons::kPixelizedSideIcon;
+  (*map)[&views::kMenuCloseCustomIcon] = &aerographene_theme_icons::kPixelizedSideIcon;
+  // i-search: toolbar/app_menu_model.cc:2267, omnibox
+  (*map)[&vector_icons::kSearchIcon] = &aerographene_theme_icons::kPixelizedSearchIcon;
+  (*map)[&vector_icons::kSearchChromeRefreshOldIcon] = &aerographene_theme_icons::kPixelizedSearchIcon;
+  // i-hist: tabs/recent_tabs_builder.cc:247-248
+  (*map)[&vector_icons::kHistoryIcon] = &aerographene_theme_icons::kPixelizedHistIcon;
+  (*map)[&::kHistoryIcon] = &aerographene_theme_icons::kPixelizedHistIcon;
+  (*map)[&vector_icons::kHistoryChromeRefreshOldIcon] = &aerographene_theme_icons::kPixelizedHistIcon;
+  // i-dl: views/download/bubble/download_toolbar_ui_controller.cc:698-702
+  (*map)[&vector_icons::kDownloadIcon] = &aerographene_theme_icons::kPixelizedDlIcon;
+  (*map)[&::kDownloadIcon] = &aerographene_theme_icons::kPixelizedDlIcon;
+  (*map)[&::kDownloadToolbarButtonChromeRefreshOldIcon] = &aerographene_theme_icons::kPixelizedDlIcon;
+  (*map)[&::kDownloadToolbarButtonTouchOldIcon] = &aerographene_theme_icons::kPixelizedDlIcon;
+  // i-ext: extensions/extensions_toolbar_view_model.cc:179-181
+  (*map)[&vector_icons::kChromeExtensionIcon] = &aerographene_theme_icons::kPixelizedExtIcon;
+  (*map)[&vector_icons::kExtensionChromeRefreshOldIcon] = &aerographene_theme_icons::kPixelizedExtIcon;
+  // i-more: views/toolbar/browser_app_menu_button.cc:154-159
+  (*map)[&::kMoreVertIcon] = &aerographene_theme_icons::kPixelizedMoreIcon;
+  (*map)[&::kBrowserToolsChromeRefreshOldIcon] = &aerographene_theme_icons::kPixelizedMoreIcon;
+  (*map)[&::kBrowserToolsTouchOldIcon] = &aerographene_theme_icons::kPixelizedMoreIcon;
+  // i-link: toolbar/app_menu_model.cc:921-922
+  (*map)[&vector_icons::kLinkIcon] = &aerographene_theme_icons::kPixelizedLinkIcon;
+  (*map)[&::kLinkChromeRefreshOldIcon] = &aerographene_theme_icons::kPixelizedLinkIcon;
+  // i-info: views/location_bar/location_icon_view.cc:208-211
+  (*map)[&omnibox::kPageInfoCustomIcon] = &aerographene_theme_icons::kPixelizedInfoIcon;
+  (*map)[&omnibox::kSecurePageInfoChromeRefreshOldIcon] = &aerographene_theme_icons::kPixelizedInfoIcon;
+  // i-plus: views/toolbar/app_menu.cc:733, omnibox/omnibox_edit_model.cc:586
+  (*map)[&::kAddIcon] = &aerographene_theme_icons::kPixelizedPlusIcon;
+  (*map)[&::kAddChromeRefreshOldIcon] = &aerographene_theme_icons::kPixelizedPlusIcon;
+  // i-close: views/tabs/tab/tab_close_button.cc:212-214, ui/views/window/frame_view_linux.cc:92-93
+  (*map)[&views::kCloseIcon] = &aerographene_theme_icons::kPixelizedCloseIcon;
+  (*map)[&vector_icons::kCloseIcon] = &aerographene_theme_icons::kPixelizedCloseIcon;
+  (*map)[&::kCloseIcon] = &aerographene_theme_icons::kPixelizedCloseIcon;
+  (*map)[&vector_icons::kCloseChromeRefreshOldIcon] = &aerographene_theme_icons::kPixelizedCloseIcon;
+  (*map)[&::kCloseChromeRefreshOldIcon] = &aerographene_theme_icons::kPixelizedCloseIcon;
+  (*map)[&::kCloseWeight500CustomIcon] = &aerographene_theme_icons::kPixelizedCloseIcon;
+  (*map)[&::kCloseTabChromeRefreshOldIcon] = &aerographene_theme_icons::kPixelizedCloseIcon;
+  (*map)[&views::kWindowControlCloseOldIcon] = &aerographene_theme_icons::kPixelizedCloseIcon;
+  // i-min: ui/views/window/frame_view_linux.cc:99-100, views/frame/opaque_browser_frame_view.cc:173
+  (*map)[&views::kChromeMinimizeIcon] = &aerographene_theme_icons::kPixelizedMinIcon;
+  (*map)[&::kChromeMinimizeIcon] = &aerographene_theme_icons::kPixelizedMinIcon;
+  (*map)[&views::kWindowControlMinimizeOldIcon] = &aerographene_theme_icons::kPixelizedMinIcon;
+  // i-max: ui/views/window/frame_view_linux.cc:106-107
+  (*map)[&views::kChromeMaximizeIcon] = &aerographene_theme_icons::kPixelizedMaxIcon;
+  (*map)[&views::kWindowControlMaximizeOldIcon] = &aerographene_theme_icons::kPixelizedMaxIcon;
+  // i-win: toolbar/app_menu_model.cc:2104
+  (*map)[&views::kNewWindowIcon] = &aerographene_theme_icons::kPixelizedWinIcon;
+  (*map)[&::kNewWindowIcon] = &aerographene_theme_icons::kPixelizedWinIcon;
+  (*map)[&views::kNewWindowOldIcon] = &aerographene_theme_icons::kPixelizedWinIcon;
+  (*map)[&::kNewWindowOldIcon] = &aerographene_theme_icons::kPixelizedWinIcon;
+  // i-minus: views/location_bar/zoom_bubble_view.cc:291, views/zoom/zoom_view_controller.cc:80
+  (*map)[&::kRemoveIcon] = &aerographene_theme_icons::kPixelizedMinusIcon;
+  (*map)[&::kRemoveOldIcon] = &aerographene_theme_icons::kPixelizedMinusIcon;
+  (*map)[&::kZoomMinusChromeRefreshOldIcon] = &aerographene_theme_icons::kPixelizedMinusIcon;
+  // i-chev: views/find_bar_view.cc:299, views/tabs/common/tab_group_header_view.cc:566
+  (*map)[&vector_icons::kKeyboardArrowDownIcon] = &aerographene_theme_icons::kPixelizedChevIcon;
+  (*map)[&::kKeyboardArrowDownIcon] = &aerographene_theme_icons::kPixelizedChevIcon;
+  (*map)[&::kKeyboardArrowDownChromeRefreshOldIcon] = &aerographene_theme_icons::kPixelizedChevIcon;
+  // i-chevr: ui/views/controls/menu/menu_item_view.cc:1705-1706
+  (*map)[&vector_icons::kKeyboardArrowRightFlippableIcon] = &aerographene_theme_icons::kPixelizedChevrIcon;
+  (*map)[&vector_icons::kSubmenuArrowChromeRefreshOldIcon] = &aerographene_theme_icons::kPixelizedChevrIcon;
+  // i-chl: views/tabs/horizontal/tab_scroll_button_container.cc:152
+  (*map)[&::kKeyboardArrowLeftIcon] = &aerographene_theme_icons::kPixelizedChlIcon;
+  // i-chr: views/tabs/horizontal/tab_scroll_button_container.cc:166, views/download/bubble/download_bubble_row_view.cc:473
+  (*map)[&::kKeyboardArrowRightIcon] = &aerographene_theme_icons::kPixelizedChrIcon;
+  (*map)[&::kChevronRightIcon] = &aerographene_theme_icons::kPixelizedChrIcon;
+  // i-up: views/tabs/shared/drop_arrow.cc:35
+  (*map)[&::kArrowUpwardIcon] = &aerographene_theme_icons::kPixelizedUpIcon;
+  // i-down: views/tabs/shared/drop_arrow.cc:38
+  (*map)[&::kArrowDownwardIcon] = &aerographene_theme_icons::kPixelizedDownIcon;
+  // i-check: ui/views/controls/menu/menu_item_view.cc:1713
+  (*map)[&views::kCheckIcon] = &aerographene_theme_icons::kPixelizedCheckIcon;
+  (*map)[&::kCheckIcon] = &aerographene_theme_icons::kPixelizedCheckIcon;
+  (*map)[&views::kMenuCheckOldIcon] = &aerographene_theme_icons::kPixelizedCheckIcon;
+  // i-gear: toolbar/app_menu_model.cc:2347, page_info/chrome_page_info_delegate.cc:268
+  (*map)[&vector_icons::kSettingsIcon] = &aerographene_theme_icons::kPixelizedGearIcon;
+  (*map)[&::kSettingsIcon] = &aerographene_theme_icons::kPixelizedGearIcon;
+  (*map)[&::kSettingsMenuOldIcon] = &aerographene_theme_icons::kPixelizedGearIcon;
+  (*map)[&vector_icons::kSettingsChromeRefreshOldIcon] = &aerographene_theme_icons::kPixelizedGearIcon;
+  // i-mon: toolbar/app_menu_model.cc:905
+  (*map)[&vector_icons::kDesktopWindowsIcon] = &aerographene_theme_icons::kPixelizedMonIcon;
+  (*map)[&::kDesktopWindowsIcon] = &aerographene_theme_icons::kPixelizedMonIcon;
+  // i-split: tabs/tab_menu_model.cc:347-349
+  (*map)[&::kSplitSceneIcon] = &aerographene_theme_icons::kPixelizedSplitIcon;
+  (*map)[&::kSplitSceneOldIcon] = &aerographene_theme_icons::kPixelizedSplitIcon;
+  // i-ext2: tabs/tab_menu_model.cc:463-464
+  (*map)[&::kOpenInNewIcon] = &aerographene_theme_icons::kPixelizedExt2Icon;
+  (*map)[&views::kOpenInNewIcon] = &aerographene_theme_icons::kPixelizedExt2Icon;
+  (*map)[&::kOpenInNewOldIcon] = &aerographene_theme_icons::kPixelizedExt2Icon;
+  // i-folder: bookmarks/bookmark_utils.cc:388-390, ui/views/controls/tree/tree_view.cc:115
+  (*map)[&vector_icons::kFolderFlippableIcon] = &aerographene_theme_icons::kPixelizedFolderIcon;
+  (*map)[&vector_icons::kFolderChromeRefreshOldIcon] = &aerographene_theme_icons::kPixelizedFolderIcon;
+  // i-bell: page_info/page_info_view_factory.cc:353
+  (*map)[&vector_icons::kNotificationsIcon] = &aerographene_theme_icons::kPixelizedBellIcon;
+  (*map)[&vector_icons::kNotificationsChromeRefreshOldIcon] = &aerographene_theme_icons::kPixelizedBellIcon;
+  // i-spk: tabs/alert/tab_alert_icon.cc:72
+  (*map)[&vector_icons::kVolumeUpIcon] = &aerographene_theme_icons::kPixelizedSpkIcon;
+  (*map)[&vector_icons::kVolumeUpChromeRefreshOldIcon] = &aerographene_theme_icons::kPixelizedSpkIcon;
+  // i-spkoff: tabs/alert/tab_alert_icon.cc:76
+  (*map)[&vector_icons::kVolumeOffIcon] = &aerographene_theme_icons::kPixelizedSpkoffIcon;
+  (*map)[&vector_icons::kVolumeOffChromeRefreshOldIcon] = &aerographene_theme_icons::kPixelizedSpkoffIcon;
 }
 
 }  // namespace
--- /dev/null
+++ b/chrome/browser/ui/aerographene_theme_icons/pixelized_back.icon
@@ -0,0 +1,62 @@
+// Copyright 2026 The aerographene Authors
+// Use of this source code is governed by the GPL-3.0 license.
+// Gerado por theming/design-language/gen_theme_icons.py; não editar.
+
+CANVAS_DIMENSIONS, 16,
+FILL_RULE_NONZERO,
+DISABLE_AA,
+MOVE_TO, 6, 2,
+LINE_TO, 7, 2,
+LINE_TO, 7, 3,
+LINE_TO, 6, 3,
+CLOSE,
+MOVE_TO, 5, 3,
+LINE_TO, 6, 3,
+LINE_TO, 6, 4,
+LINE_TO, 5, 4,
+CLOSE,
+MOVE_TO, 4, 4,
+LINE_TO, 5, 4,
+LINE_TO, 5, 5,
+LINE_TO, 4, 5,
+CLOSE,
+MOVE_TO, 3, 5,
+LINE_TO, 4, 5,
+LINE_TO, 4, 6,
+LINE_TO, 3, 6,
+CLOSE,
+MOVE_TO, 2, 6,
+LINE_TO, 3, 6,
+LINE_TO, 3, 7,
+LINE_TO, 2, 7,
+CLOSE,
+MOVE_TO, 1, 7,
+LINE_TO, 14, 7,
+LINE_TO, 14, 8,
+LINE_TO, 1, 8,
+CLOSE,
+MOVE_TO, 2, 8,
+LINE_TO, 3, 8,
+LINE_TO, 3, 9,
+LINE_TO, 2, 9,
+CLOSE,
+MOVE_TO, 3, 9,
+LINE_TO, 4, 9,
+LINE_TO, 4, 10,
+LINE_TO, 3, 10,
+CLOSE,
+MOVE_TO, 4, 10,
+LINE_TO, 5, 10,
+LINE_TO, 5, 11,
+LINE_TO, 4, 11,
+CLOSE,
+MOVE_TO, 5, 11,
+LINE_TO, 6, 11,
+LINE_TO, 6, 12,
+LINE_TO, 5, 12,
+CLOSE,
+MOVE_TO, 6, 12,
+LINE_TO, 7, 12,
+LINE_TO, 7, 13,
+LINE_TO, 6, 13,
+CLOSE,
--- /dev/null
+++ b/chrome/browser/ui/aerographene_theme_icons/pixelized_bell.icon
@@ -0,0 +1,165 @@
+// Copyright 2026 The aerographene Authors
+// Use of this source code is governed by the GPL-3.0 license.
+// Gerado por theming/design-language/gen_theme_icons.py; não editar.
+
+CANVAS_DIMENSIONS, 16,
+PATH_COLOR_ALPHA, 0x38,
+FILL_RULE_NONZERO,
+DISABLE_AA,
+MOVE_TO, 6, 4,
+LINE_TO, 9, 4,
+LINE_TO, 9, 5,
+LINE_TO, 6, 5,
+CLOSE,
+MOVE_TO, 5, 5,
+LINE_TO, 10, 5,
+LINE_TO, 10, 6,
+LINE_TO, 5, 6,
+CLOSE,
+MOVE_TO, 5, 6,
+LINE_TO, 10, 6,
+LINE_TO, 10, 7,
+LINE_TO, 5, 7,
+CLOSE,
+MOVE_TO, 5, 7,
+LINE_TO, 10, 7,
+LINE_TO, 10, 8,
+LINE_TO, 5, 8,
+CLOSE,
+MOVE_TO, 4, 8,
+LINE_TO, 11, 8,
+LINE_TO, 11, 9,
+LINE_TO, 4, 9,
+CLOSE,
+MOVE_TO, 4, 9,
+LINE_TO, 11, 9,
+LINE_TO, 11, 10,
+LINE_TO, 4, 10,
+CLOSE,
+MOVE_TO, 4, 10,
+LINE_TO, 11, 10,
+LINE_TO, 11, 11,
+LINE_TO, 4, 11,
+CLOSE,
+NEW_PATH,
+PATH_COLOR_ALPHA, 0x80,
+FILL_RULE_NONZERO,
+DISABLE_AA,
+MOVE_TO, 11, 5,
+LINE_TO, 12, 5,
+LINE_TO, 12, 6,
+LINE_TO, 11, 6,
+CLOSE,
+MOVE_TO, 11, 6,
+LINE_TO, 12, 6,
+LINE_TO, 12, 7,
+LINE_TO, 11, 7,
+CLOSE,
+MOVE_TO, 12, 8,
+LINE_TO, 13, 8,
+LINE_TO, 13, 9,
+LINE_TO, 12, 9,
+CLOSE,
+MOVE_TO, 12, 9,
+LINE_TO, 13, 9,
+LINE_TO, 13, 10,
+LINE_TO, 12, 10,
+CLOSE,
+MOVE_TO, 4, 12,
+LINE_TO, 6, 12,
+LINE_TO, 6, 13,
+LINE_TO, 4, 13,
+CLOSE,
+MOVE_TO, 9, 12,
+LINE_TO, 11, 12,
+LINE_TO, 11, 13,
+LINE_TO, 9, 13,
+CLOSE,
+NEW_PATH,
+FILL_RULE_NONZERO,
+DISABLE_AA,
+MOVE_TO, 6, 3,
+LINE_TO, 9, 3,
+LINE_TO, 9, 4,
+LINE_TO, 6, 4,
+CLOSE,
+MOVE_TO, 4, 4,
+LINE_TO, 6, 4,
+LINE_TO, 6, 5,
+LINE_TO, 4, 5,
+CLOSE,
+MOVE_TO, 9, 4,
+LINE_TO, 11, 4,
+LINE_TO, 11, 5,
+LINE_TO, 9, 5,
+CLOSE,
+MOVE_TO, 4, 5,
+LINE_TO, 5, 5,
+LINE_TO, 5, 6,
+LINE_TO, 4, 6,
+CLOSE,
+MOVE_TO, 10, 5,
+LINE_TO, 11, 5,
+LINE_TO, 11, 6,
+LINE_TO, 10, 6,
+CLOSE,
+MOVE_TO, 4, 6,
+LINE_TO, 5, 6,
+LINE_TO, 5, 7,
+LINE_TO, 4, 7,
+CLOSE,
+MOVE_TO, 10, 6,
+LINE_TO, 11, 6,
+LINE_TO, 11, 7,
+LINE_TO, 10, 7,
+CLOSE,
+MOVE_TO, 3, 7,
+LINE_TO, 5, 7,
+LINE_TO, 5, 8,
+LINE_TO, 3, 8,
+CLOSE,
+MOVE_TO, 10, 7,
+LINE_TO, 12, 7,
+LINE_TO, 12, 8,
+LINE_TO, 10, 8,
+CLOSE,
+MOVE_TO, 3, 8,
+LINE_TO, 4, 8,
+LINE_TO, 4, 9,
+LINE_TO, 3, 9,
+CLOSE,
+MOVE_TO, 11, 8,
+LINE_TO, 12, 8,
+LINE_TO, 12, 9,
+LINE_TO, 11, 9,
+CLOSE,
+MOVE_TO, 3, 9,
+LINE_TO, 4, 9,
+LINE_TO, 4, 10,
+LINE_TO, 3, 10,
+CLOSE,
+MOVE_TO, 11, 9,
+LINE_TO, 12, 9,
+LINE_TO, 12, 10,
+LINE_TO, 11, 10,
+CLOSE,
+MOVE_TO, 2, 10,
+LINE_TO, 4, 10,
+LINE_TO, 4, 11,
+LINE_TO, 2, 11,
+CLOSE,
+MOVE_TO, 11, 10,
+LINE_TO, 13, 10,
+LINE_TO, 13, 11,
+LINE_TO, 11, 11,
+CLOSE,
+MOVE_TO, 2, 11,
+LINE_TO, 13, 11,
+LINE_TO, 13, 12,
+LINE_TO, 2, 12,
+CLOSE,
+MOVE_TO, 6, 12,
+LINE_TO, 9, 12,
+LINE_TO, 9, 13,
+LINE_TO, 6, 13,
+CLOSE,
--- /dev/null
+++ b/chrome/browser/ui/aerographene_theme_icons/pixelized_check.icon
@@ -0,0 +1,67 @@
+// Copyright 2026 The aerographene Authors
+// Use of this source code is governed by the GPL-3.0 license.
+// Gerado por theming/design-language/gen_theme_icons.py; não editar.
+
+CANVAS_DIMENSIONS, 16,
+FILL_RULE_NONZERO,
+DISABLE_AA,
+MOVE_TO, 13, 3,
+LINE_TO, 14, 3,
+LINE_TO, 14, 4,
+LINE_TO, 13, 4,
+CLOSE,
+MOVE_TO, 12, 4,
+LINE_TO, 13, 4,
+LINE_TO, 13, 5,
+LINE_TO, 12, 5,
+CLOSE,
+MOVE_TO, 11, 5,
+LINE_TO, 12, 5,
+LINE_TO, 12, 6,
+LINE_TO, 11, 6,
+CLOSE,
+MOVE_TO, 10, 6,
+LINE_TO, 11, 6,
+LINE_TO, 11, 7,
+LINE_TO, 10, 7,
+CLOSE,
+MOVE_TO, 9, 7,
+LINE_TO, 10, 7,
+LINE_TO, 10, 8,
+LINE_TO, 9, 8,
+CLOSE,
+MOVE_TO, 2, 8,
+LINE_TO, 3, 8,
+LINE_TO, 3, 9,
+LINE_TO, 2, 9,
+CLOSE,
+MOVE_TO, 8, 8,
+LINE_TO, 9, 8,
+LINE_TO, 9, 9,
+LINE_TO, 8, 9,
+CLOSE,
+MOVE_TO, 3, 9,
+LINE_TO, 4, 9,
+LINE_TO, 4, 10,
+LINE_TO, 3, 10,
+CLOSE,
+MOVE_TO, 7, 9,
+LINE_TO, 8, 9,
+LINE_TO, 8, 10,
+LINE_TO, 7, 10,
+CLOSE,
+MOVE_TO, 4, 10,
+LINE_TO, 5, 10,
+LINE_TO, 5, 11,
+LINE_TO, 4, 11,
+CLOSE,
+MOVE_TO, 6, 10,
+LINE_TO, 7, 10,
+LINE_TO, 7, 11,
+LINE_TO, 6, 11,
+CLOSE,
+MOVE_TO, 5, 11,
+LINE_TO, 6, 11,
+LINE_TO, 6, 12,
+LINE_TO, 5, 12,
+CLOSE,
--- /dev/null
+++ b/chrome/browser/ui/aerographene_theme_icons/pixelized_chev.icon
@@ -0,0 +1,62 @@
+// Copyright 2026 The aerographene Authors
+// Use of this source code is governed by the GPL-3.0 license.
+// Gerado por theming/design-language/gen_theme_icons.py; não editar.
+
+CANVAS_DIMENSIONS, 16,
+FILL_RULE_NONZERO,
+DISABLE_AA,
+MOVE_TO, 2, 5,
+LINE_TO, 3, 5,
+LINE_TO, 3, 6,
+LINE_TO, 2, 6,
+CLOSE,
+MOVE_TO, 12, 5,
+LINE_TO, 13, 5,
+LINE_TO, 13, 6,
+LINE_TO, 12, 6,
+CLOSE,
+MOVE_TO, 3, 6,
+LINE_TO, 4, 6,
+LINE_TO, 4, 7,
+LINE_TO, 3, 7,
+CLOSE,
+MOVE_TO, 11, 6,
+LINE_TO, 12, 6,
+LINE_TO, 12, 7,
+LINE_TO, 11, 7,
+CLOSE,
+MOVE_TO, 4, 7,
+LINE_TO, 5, 7,
+LINE_TO, 5, 8,
+LINE_TO, 4, 8,
+CLOSE,
+MOVE_TO, 10, 7,
+LINE_TO, 11, 7,
+LINE_TO, 11, 8,
+LINE_TO, 10, 8,
+CLOSE,
+MOVE_TO, 5, 8,
+LINE_TO, 6, 8,
+LINE_TO, 6, 9,
+LINE_TO, 5, 9,
+CLOSE,
+MOVE_TO, 9, 8,
+LINE_TO, 10, 8,
+LINE_TO, 10, 9,
+LINE_TO, 9, 9,
+CLOSE,
+MOVE_TO, 6, 9,
+LINE_TO, 7, 9,
+LINE_TO, 7, 10,
+LINE_TO, 6, 10,
+CLOSE,
+MOVE_TO, 8, 9,
+LINE_TO, 9, 9,
+LINE_TO, 9, 10,
+LINE_TO, 8, 10,
+CLOSE,
+MOVE_TO, 7, 10,
+LINE_TO, 8, 10,
+LINE_TO, 8, 11,
+LINE_TO, 7, 11,
+CLOSE,
--- /dev/null
+++ b/chrome/browser/ui/aerographene_theme_icons/pixelized_chevr.icon
@@ -0,0 +1,63 @@
+// Copyright 2026 The aerographene Authors
+// Use of this source code is governed by the GPL-3.0 license.
+// Gerado por theming/design-language/gen_theme_icons.py; não editar.
+
+CANVAS_DIMENSIONS, 16,
+FLIPS_IN_RTL,
+FILL_RULE_NONZERO,
+DISABLE_AA,
+MOVE_TO, 5, 2,
+LINE_TO, 6, 2,
+LINE_TO, 6, 3,
+LINE_TO, 5, 3,
+CLOSE,
+MOVE_TO, 6, 3,
+LINE_TO, 7, 3,
+LINE_TO, 7, 4,
+LINE_TO, 6, 4,
+CLOSE,
+MOVE_TO, 7, 4,
+LINE_TO, 8, 4,
+LINE_TO, 8, 5,
+LINE_TO, 7, 5,
+CLOSE,
+MOVE_TO, 8, 5,
+LINE_TO, 9, 5,
+LINE_TO, 9, 6,
+LINE_TO, 8, 6,
+CLOSE,
+MOVE_TO, 9, 6,
+LINE_TO, 10, 6,
+LINE_TO, 10, 7,
+LINE_TO, 9, 7,
+CLOSE,
+MOVE_TO, 10, 7,
+LINE_TO, 11, 7,
+LINE_TO, 11, 8,
+LINE_TO, 10, 8,
+CLOSE,
+MOVE_TO, 9, 8,
+LINE_TO, 10, 8,
+LINE_TO, 10, 9,
+LINE_TO, 9, 9,
+CLOSE,
+MOVE_TO, 8, 9,
+LINE_TO, 9, 9,
+LINE_TO, 9, 10,
+LINE_TO, 8, 10,
+CLOSE,
+MOVE_TO, 7, 10,
+LINE_TO, 8, 10,
+LINE_TO, 8, 11,
+LINE_TO, 7, 11,
+CLOSE,
+MOVE_TO, 6, 11,
+LINE_TO, 7, 11,
+LINE_TO, 7, 12,
+LINE_TO, 6, 12,
+CLOSE,
+MOVE_TO, 5, 12,
+LINE_TO, 6, 12,
+LINE_TO, 6, 13,
+LINE_TO, 5, 13,
+CLOSE,
--- /dev/null
+++ b/chrome/browser/ui/aerographene_theme_icons/pixelized_chl.icon
@@ -0,0 +1,72 @@
+// Copyright 2026 The aerographene Authors
+// Use of this source code is governed by the GPL-3.0 license.
+// Gerado por theming/design-language/gen_theme_icons.py; não editar.
+
+CANVAS_DIMENSIONS, 16,
+FILL_RULE_NONZERO,
+DISABLE_AA,
+MOVE_TO, 10, 1,
+LINE_TO, 11, 1,
+LINE_TO, 11, 2,
+LINE_TO, 10, 2,
+CLOSE,
+MOVE_TO, 9, 2,
+LINE_TO, 10, 2,
+LINE_TO, 10, 3,
+LINE_TO, 9, 3,
+CLOSE,
+MOVE_TO, 8, 3,
+LINE_TO, 9, 3,
+LINE_TO, 9, 4,
+LINE_TO, 8, 4,
+CLOSE,
+MOVE_TO, 7, 4,
+LINE_TO, 8, 4,
+LINE_TO, 8, 5,
+LINE_TO, 7, 5,
+CLOSE,
+MOVE_TO, 6, 5,
+LINE_TO, 7, 5,
+LINE_TO, 7, 6,
+LINE_TO, 6, 6,
+CLOSE,
+MOVE_TO, 5, 6,
+LINE_TO, 6, 6,
+LINE_TO, 6, 7,
+LINE_TO, 5, 7,
+CLOSE,
+MOVE_TO, 4, 7,
+LINE_TO, 5, 7,
+LINE_TO, 5, 8,
+LINE_TO, 4, 8,
+CLOSE,
+MOVE_TO, 5, 8,
+LINE_TO, 6, 8,
+LINE_TO, 6, 9,
+LINE_TO, 5, 9,
+CLOSE,
+MOVE_TO, 6, 9,
+LINE_TO, 7, 9,
+LINE_TO, 7, 10,
+LINE_TO, 6, 10,
+CLOSE,
+MOVE_TO, 7, 10,
+LINE_TO, 8, 10,
+LINE_TO, 8, 11,
+LINE_TO, 7, 11,
+CLOSE,
+MOVE_TO, 8, 11,
+LINE_TO, 9, 11,
+LINE_TO, 9, 12,
+LINE_TO, 8, 12,
+CLOSE,
+MOVE_TO, 9, 12,
+LINE_TO, 10, 12,
+LINE_TO, 10, 13,
+LINE_TO, 9, 13,
+CLOSE,
+MOVE_TO, 10, 13,
+LINE_TO, 11, 13,
+LINE_TO, 11, 14,
+LINE_TO, 10, 14,
+CLOSE,
--- /dev/null
+++ b/chrome/browser/ui/aerographene_theme_icons/pixelized_chr.icon
@@ -0,0 +1,72 @@
+// Copyright 2026 The aerographene Authors
+// Use of this source code is governed by the GPL-3.0 license.
+// Gerado por theming/design-language/gen_theme_icons.py; não editar.
+
+CANVAS_DIMENSIONS, 16,
+FILL_RULE_NONZERO,
+DISABLE_AA,
+MOVE_TO, 4, 1,
+LINE_TO, 5, 1,
+LINE_TO, 5, 2,
+LINE_TO, 4, 2,
+CLOSE,
+MOVE_TO, 5, 2,
+LINE_TO, 6, 2,
+LINE_TO, 6, 3,
+LINE_TO, 5, 3,
+CLOSE,
+MOVE_TO, 6, 3,
+LINE_TO, 7, 3,
+LINE_TO, 7, 4,
+LINE_TO, 6, 4,
+CLOSE,
+MOVE_TO, 7, 4,
+LINE_TO, 8, 4,
+LINE_TO, 8, 5,
+LINE_TO, 7, 5,
+CLOSE,
+MOVE_TO, 8, 5,
+LINE_TO, 9, 5,
+LINE_TO, 9, 6,
+LINE_TO, 8, 6,
+CLOSE,
+MOVE_TO, 9, 6,
+LINE_TO, 10, 6,
+LINE_TO, 10, 7,
+LINE_TO, 9, 7,
+CLOSE,
+MOVE_TO, 10, 7,
+LINE_TO, 11, 7,
+LINE_TO, 11, 8,
+LINE_TO, 10, 8,
+CLOSE,
+MOVE_TO, 9, 8,
+LINE_TO, 10, 8,
+LINE_TO, 10, 9,
+LINE_TO, 9, 9,
+CLOSE,
+MOVE_TO, 8, 9,
+LINE_TO, 9, 9,
+LINE_TO, 9, 10,
+LINE_TO, 8, 10,
+CLOSE,
+MOVE_TO, 7, 10,
+LINE_TO, 8, 10,
+LINE_TO, 8, 11,
+LINE_TO, 7, 11,
+CLOSE,
+MOVE_TO, 6, 11,
+LINE_TO, 7, 11,
+LINE_TO, 7, 12,
+LINE_TO, 6, 12,
+CLOSE,
+MOVE_TO, 5, 12,
+LINE_TO, 6, 12,
+LINE_TO, 6, 13,
+LINE_TO, 5, 13,
+CLOSE,
+MOVE_TO, 4, 13,
+LINE_TO, 5, 13,
+LINE_TO, 5, 14,
+LINE_TO, 4, 14,
+CLOSE,
--- /dev/null
+++ b/chrome/browser/ui/aerographene_theme_icons/pixelized_close.icon
@@ -0,0 +1,112 @@
+// Copyright 2026 The aerographene Authors
+// Use of this source code is governed by the GPL-3.0 license.
+// Gerado por theming/design-language/gen_theme_icons.py; não editar.
+
+CANVAS_DIMENSIONS, 16,
+FILL_RULE_NONZERO,
+DISABLE_AA,
+MOVE_TO, 2, 2,
+LINE_TO, 3, 2,
+LINE_TO, 3, 3,
+LINE_TO, 2, 3,
+CLOSE,
+MOVE_TO, 12, 2,
+LINE_TO, 13, 2,
+LINE_TO, 13, 3,
+LINE_TO, 12, 3,
+CLOSE,
+MOVE_TO, 3, 3,
+LINE_TO, 4, 3,
+LINE_TO, 4, 4,
+LINE_TO, 3, 4,
+CLOSE,
+MOVE_TO, 11, 3,
+LINE_TO, 12, 3,
+LINE_TO, 12, 4,
+LINE_TO, 11, 4,
+CLOSE,
+MOVE_TO, 4, 4,
+LINE_TO, 5, 4,
+LINE_TO, 5, 5,
+LINE_TO, 4, 5,
+CLOSE,
+MOVE_TO, 10, 4,
+LINE_TO, 11, 4,
+LINE_TO, 11, 5,
+LINE_TO, 10, 5,
+CLOSE,
+MOVE_TO, 5, 5,
+LINE_TO, 6, 5,
+LINE_TO, 6, 6,
+LINE_TO, 5, 6,
+CLOSE,
+MOVE_TO, 9, 5,
+LINE_TO, 10, 5,
+LINE_TO, 10, 6,
+LINE_TO, 9, 6,
+CLOSE,
+MOVE_TO, 6, 6,
+LINE_TO, 7, 6,
+LINE_TO, 7, 7,
+LINE_TO, 6, 7,
+CLOSE,
+MOVE_TO, 8, 6,
+LINE_TO, 9, 6,
+LINE_TO, 9, 7,
+LINE_TO, 8, 7,
+CLOSE,
+MOVE_TO, 7, 7,
+LINE_TO, 8, 7,
+LINE_TO, 8, 8,
+LINE_TO, 7, 8,
+CLOSE,
+MOVE_TO, 6, 8,
+LINE_TO, 7, 8,
+LINE_TO, 7, 9,
+LINE_TO, 6, 9,
+CLOSE,
+MOVE_TO, 8, 8,
+LINE_TO, 9, 8,
+LINE_TO, 9, 9,
+LINE_TO, 8, 9,
+CLOSE,
+MOVE_TO, 5, 9,
+LINE_TO, 6, 9,
+LINE_TO, 6, 10,
+LINE_TO, 5, 10,
+CLOSE,
+MOVE_TO, 9, 9,
+LINE_TO, 10, 9,
+LINE_TO, 10, 10,
+LINE_TO, 9, 10,
+CLOSE,
+MOVE_TO, 4, 10,
+LINE_TO, 5, 10,
+LINE_TO, 5, 11,
+LINE_TO, 4, 11,
+CLOSE,
+MOVE_TO, 10, 10,
+LINE_TO, 11, 10,
+LINE_TO, 11, 11,
+LINE_TO, 10, 11,
+CLOSE,
+MOVE_TO, 3, 11,
+LINE_TO, 4, 11,
+LINE_TO, 4, 12,
+LINE_TO, 3, 12,
+CLOSE,
+MOVE_TO, 11, 11,
+LINE_TO, 12, 11,
+LINE_TO, 12, 12,
+LINE_TO, 11, 12,
+CLOSE,
+MOVE_TO, 2, 12,
+LINE_TO, 3, 12,
+LINE_TO, 3, 13,
+LINE_TO, 2, 13,
+CLOSE,
+MOVE_TO, 12, 12,
+LINE_TO, 13, 12,
+LINE_TO, 13, 13,
+LINE_TO, 12, 13,
+CLOSE,
--- /dev/null
+++ b/chrome/browser/ui/aerographene_theme_icons/pixelized_dl.icon
@@ -0,0 +1,117 @@
+// Copyright 2026 The aerographene Authors
+// Use of this source code is governed by the GPL-3.0 license.
+// Gerado por theming/design-language/gen_theme_icons.py; não editar.
+
+CANVAS_DIMENSIONS, 16,
+FILL_RULE_NONZERO,
+DISABLE_AA,
+MOVE_TO, 7, 1,
+LINE_TO, 8, 1,
+LINE_TO, 8, 2,
+LINE_TO, 7, 2,
+CLOSE,
+MOVE_TO, 7, 2,
+LINE_TO, 8, 2,
+LINE_TO, 8, 3,
+LINE_TO, 7, 3,
+CLOSE,
+MOVE_TO, 7, 3,
+LINE_TO, 8, 3,
+LINE_TO, 8, 4,
+LINE_TO, 7, 4,
+CLOSE,
+MOVE_TO, 7, 4,
+LINE_TO, 8, 4,
+LINE_TO, 8, 5,
+LINE_TO, 7, 5,
+CLOSE,
+MOVE_TO, 3, 5,
+LINE_TO, 4, 5,
+LINE_TO, 4, 6,
+LINE_TO, 3, 6,
+CLOSE,
+MOVE_TO, 7, 5,
+LINE_TO, 8, 5,
+LINE_TO, 8, 6,
+LINE_TO, 7, 6,
+CLOSE,
+MOVE_TO, 11, 5,
+LINE_TO, 12, 5,
+LINE_TO, 12, 6,
+LINE_TO, 11, 6,
+CLOSE,
+MOVE_TO, 4, 6,
+LINE_TO, 5, 6,
+LINE_TO, 5, 7,
+LINE_TO, 4, 7,
+CLOSE,
+MOVE_TO, 7, 6,
+LINE_TO, 8, 6,
+LINE_TO, 8, 7,
+LINE_TO, 7, 7,
+CLOSE,
+MOVE_TO, 10, 6,
+LINE_TO, 11, 6,
+LINE_TO, 11, 7,
+LINE_TO, 10, 7,
+CLOSE,
+MOVE_TO, 5, 7,
+LINE_TO, 6, 7,
+LINE_TO, 6, 8,
+LINE_TO, 5, 8,
+CLOSE,
+MOVE_TO, 7, 7,
+LINE_TO, 8, 7,
+LINE_TO, 8, 8,
+LINE_TO, 7, 8,
+CLOSE,
+MOVE_TO, 9, 7,
+LINE_TO, 10, 7,
+LINE_TO, 10, 8,
+LINE_TO, 9, 8,
+CLOSE,
+MOVE_TO, 6, 8,
+LINE_TO, 9, 8,
+LINE_TO, 9, 9,
+LINE_TO, 6, 9,
+CLOSE,
+MOVE_TO, 7, 9,
+LINE_TO, 8, 9,
+LINE_TO, 8, 10,
+LINE_TO, 7, 10,
+CLOSE,
+MOVE_TO, 1, 10,
+LINE_TO, 2, 10,
+LINE_TO, 2, 11,
+LINE_TO, 1, 11,
+CLOSE,
+MOVE_TO, 13, 10,
+LINE_TO, 14, 10,
+LINE_TO, 14, 11,
+LINE_TO, 13, 11,
+CLOSE,
+MOVE_TO, 1, 11,
+LINE_TO, 2, 11,
+LINE_TO, 2, 12,
+LINE_TO, 1, 12,
+CLOSE,
+MOVE_TO, 13, 11,
+LINE_TO, 14, 11,
+LINE_TO, 14, 12,
+LINE_TO, 13, 12,
+CLOSE,
+MOVE_TO, 1, 12,
+LINE_TO, 2, 12,
+LINE_TO, 2, 13,
+LINE_TO, 1, 13,
+CLOSE,
+MOVE_TO, 13, 12,
+LINE_TO, 14, 12,
+LINE_TO, 14, 13,
+LINE_TO, 13, 13,
+CLOSE,
+MOVE_TO, 1, 13,
+LINE_TO, 14, 13,
+LINE_TO, 14, 14,
+LINE_TO, 1, 14,
+CLOSE,
--- /dev/null
+++ b/chrome/browser/ui/aerographene_theme_icons/pixelized_down.icon
@@ -0,0 +1,112 @@
+// Copyright 2026 The aerographene Authors
+// Use of this source code is governed by the GPL-3.0 license.
+// Gerado por theming/design-language/gen_theme_icons.py; não editar.
+
+CANVAS_DIMENSIONS, 16,
+FILL_RULE_NONZERO,
+DISABLE_AA,
+MOVE_TO, 7, 1,
+LINE_TO, 8, 1,
+LINE_TO, 8, 2,
+LINE_TO, 7, 2,
+CLOSE,
+MOVE_TO, 7, 2,
+LINE_TO, 8, 2,
+LINE_TO, 8, 3,
+LINE_TO, 7, 3,
+CLOSE,
+MOVE_TO, 7, 3,
+LINE_TO, 8, 3,
+LINE_TO, 8, 4,
+LINE_TO, 7, 4,
+CLOSE,
+MOVE_TO, 7, 4,
+LINE_TO, 8, 4,
+LINE_TO, 8, 5,
+LINE_TO, 7, 5,
+CLOSE,
+MOVE_TO, 7, 5,
+LINE_TO, 8, 5,
+LINE_TO, 8, 6,
+LINE_TO, 7, 6,
+CLOSE,
+MOVE_TO, 7, 6,
+LINE_TO, 8, 6,
+LINE_TO, 8, 7,
+LINE_TO, 7, 7,
+CLOSE,
+MOVE_TO, 7, 7,
+LINE_TO, 8, 7,
+LINE_TO, 8, 8,
+LINE_TO, 7, 8,
+CLOSE,
+MOVE_TO, 2, 8,
+LINE_TO, 3, 8,
+LINE_TO, 3, 9,
+LINE_TO, 2, 9,
+CLOSE,
+MOVE_TO, 7, 8,
+LINE_TO, 8, 8,
+LINE_TO, 8, 9,
+LINE_TO, 7, 9,
+CLOSE,
+MOVE_TO, 12, 8,
+LINE_TO, 13, 8,
+LINE_TO, 13, 9,
+LINE_TO, 12, 9,
+CLOSE,
+MOVE_TO, 3, 9,
+LINE_TO, 4, 9,
+LINE_TO, 4, 10,
+LINE_TO, 3, 10,
+CLOSE,
+MOVE_TO, 7, 9,
+LINE_TO, 8, 9,
+LINE_TO, 8, 10,
+LINE_TO, 7, 10,
+CLOSE,
+MOVE_TO, 11, 9,
+LINE_TO, 12, 9,
+LINE_TO, 12, 10,
+LINE_TO, 11, 10,
+CLOSE,
+MOVE_TO, 4, 10,
+LINE_TO, 5, 10,
+LINE_TO, 5, 11,
+LINE_TO, 4, 11,
+CLOSE,
+MOVE_TO, 7, 10,
+LINE_TO, 8, 10,
+LINE_TO, 8, 11,
+LINE_TO, 7, 11,
+CLOSE,
+MOVE_TO, 10, 10,
+LINE_TO, 11, 10,
+LINE_TO, 11, 11,
+LINE_TO, 10, 11,
+CLOSE,
+MOVE_TO, 5, 11,
+LINE_TO, 6, 11,
+LINE_TO, 6, 12,
+LINE_TO, 5, 12,
+CLOSE,
+MOVE_TO, 7, 11,
+LINE_TO, 8, 11,
+LINE_TO, 8, 12,
+LINE_TO, 7, 12,
+CLOSE,
+MOVE_TO, 9, 11,
+LINE_TO, 10, 11,
+LINE_TO, 10, 12,
+LINE_TO, 9, 12,
+CLOSE,
+MOVE_TO, 6, 12,
+LINE_TO, 9, 12,
+LINE_TO, 9, 13,
+LINE_TO, 6, 13,
+CLOSE,
+MOVE_TO, 7, 13,
+LINE_TO, 8, 13,
+LINE_TO, 8, 14,
+LINE_TO, 7, 14,
+CLOSE,
--- /dev/null
+++ b/chrome/browser/ui/aerographene_theme_icons/pixelized_ext.icon
@@ -0,0 +1,355 @@
+// Copyright 2026 The aerographene Authors
+// Use of this source code is governed by the GPL-3.0 license.
+// Gerado por theming/design-language/gen_theme_icons.py; não editar.
+
+CANVAS_DIMENSIONS, 16,
+PATH_COLOR_ALPHA, 0x38,
+FILL_RULE_NONZERO,
+DISABLE_AA,
+MOVE_TO, 2, 2,
+LINE_TO, 6, 2,
+LINE_TO, 6, 3,
+LINE_TO, 2, 3,
+CLOSE,
+MOVE_TO, 10, 2,
+LINE_TO, 13, 2,
+LINE_TO, 13, 3,
+LINE_TO, 10, 3,
+CLOSE,
+MOVE_TO, 2, 3,
+LINE_TO, 6, 3,
+LINE_TO, 6, 4,
+LINE_TO, 2, 4,
+CLOSE,
+MOVE_TO, 10, 3,
+LINE_TO, 13, 3,
+LINE_TO, 13, 4,
+LINE_TO, 10, 4,
+CLOSE,
+MOVE_TO, 2, 4,
+LINE_TO, 6, 4,
+LINE_TO, 6, 5,
+LINE_TO, 2, 5,
+CLOSE,
+MOVE_TO, 10, 4,
+LINE_TO, 13, 4,
+LINE_TO, 13, 5,
+LINE_TO, 10, 5,
+CLOSE,
+MOVE_TO, 2, 8,
+LINE_TO, 6, 8,
+LINE_TO, 6, 9,
+LINE_TO, 2, 9,
+CLOSE,
+MOVE_TO, 9, 8,
+LINE_TO, 13, 8,
+LINE_TO, 13, 9,
+LINE_TO, 9, 9,
+CLOSE,
+MOVE_TO, 2, 9,
+LINE_TO, 6, 9,
+LINE_TO, 6, 10,
+LINE_TO, 2, 10,
+CLOSE,
+MOVE_TO, 9, 9,
+LINE_TO, 13, 9,
+LINE_TO, 13, 10,
+LINE_TO, 9, 10,
+CLOSE,
+MOVE_TO, 2, 10,
+LINE_TO, 6, 10,
+LINE_TO, 6, 11,
+LINE_TO, 2, 11,
+CLOSE,
+MOVE_TO, 9, 10,
+LINE_TO, 13, 10,
+LINE_TO, 13, 11,
+LINE_TO, 9, 11,
+CLOSE,
+MOVE_TO, 2, 11,
+LINE_TO, 6, 11,
+LINE_TO, 6, 12,
+LINE_TO, 2, 12,
+CLOSE,
+MOVE_TO, 9, 11,
+LINE_TO, 13, 11,
+LINE_TO, 13, 12,
+LINE_TO, 9, 12,
+CLOSE,
+NEW_PATH,
+PATH_COLOR_ALPHA, 0x80,
+FILL_RULE_NONZERO,
+DISABLE_AA,
+MOVE_TO, 7, 2,
+LINE_TO, 8, 2,
+LINE_TO, 8, 3,
+LINE_TO, 7, 3,
+CLOSE,
+MOVE_TO, 14, 2,
+LINE_TO, 15, 2,
+LINE_TO, 15, 3,
+LINE_TO, 14, 3,
+CLOSE,
+MOVE_TO, 7, 3,
+LINE_TO, 8, 3,
+LINE_TO, 8, 4,
+LINE_TO, 7, 4,
+CLOSE,
+MOVE_TO, 14, 3,
+LINE_TO, 15, 3,
+LINE_TO, 15, 4,
+LINE_TO, 14, 4,
+CLOSE,
+MOVE_TO, 7, 4,
+LINE_TO, 8, 4,
+LINE_TO, 8, 5,
+LINE_TO, 7, 5,
+CLOSE,
+MOVE_TO, 14, 4,
+LINE_TO, 15, 4,
+LINE_TO, 15, 5,
+LINE_TO, 14, 5,
+CLOSE,
+MOVE_TO, 2, 6,
+LINE_TO, 6, 6,
+LINE_TO, 6, 7,
+LINE_TO, 2, 7,
+CLOSE,
+MOVE_TO, 10, 6,
+LINE_TO, 13, 6,
+LINE_TO, 13, 7,
+LINE_TO, 10, 7,
+CLOSE,
+MOVE_TO, 7, 8,
+LINE_TO, 8, 8,
+LINE_TO, 8, 9,
+LINE_TO, 7, 9,
+CLOSE,
+MOVE_TO, 14, 8,
+LINE_TO, 15, 8,
+LINE_TO, 15, 9,
+LINE_TO, 14, 9,
+CLOSE,
+MOVE_TO, 7, 9,
+LINE_TO, 8, 9,
+LINE_TO, 8, 10,
+LINE_TO, 7, 10,
+CLOSE,
+MOVE_TO, 14, 9,
+LINE_TO, 15, 9,
+LINE_TO, 15, 10,
+LINE_TO, 14, 10,
+CLOSE,
+MOVE_TO, 7, 10,
+LINE_TO, 8, 10,
+LINE_TO, 8, 11,
+LINE_TO, 7, 11,
+CLOSE,
+MOVE_TO, 14, 10,
+LINE_TO, 15, 10,
+LINE_TO, 15, 11,
+LINE_TO, 14, 11,
+CLOSE,
+MOVE_TO, 7, 11,
+LINE_TO, 8, 11,
+LINE_TO, 8, 12,
+LINE_TO, 7, 12,
+CLOSE,
+MOVE_TO, 14, 11,
+LINE_TO, 15, 11,
+LINE_TO, 15, 12,
+LINE_TO, 14, 12,
+CLOSE,
+MOVE_TO, 2, 13,
+LINE_TO, 6, 13,
+LINE_TO, 6, 14,
+LINE_TO, 2, 14,
+CLOSE,
+MOVE_TO, 9, 13,
+LINE_TO, 13, 13,
+LINE_TO, 13, 14,
+LINE_TO, 9, 14,
+CLOSE,
+NEW_PATH,
+FILL_RULE_NONZERO,
+DISABLE_AA,
+MOVE_TO, 1, 1,
+LINE_TO, 7, 1,
+LINE_TO, 7, 2,
+LINE_TO, 1, 2,
+CLOSE,
+MOVE_TO, 9, 1,
+LINE_TO, 14, 1,
+LINE_TO, 14, 2,
+LINE_TO, 9, 2,
+CLOSE,
+MOVE_TO, 1, 2,
+LINE_TO, 2, 2,
+LINE_TO, 2, 3,
+LINE_TO, 1, 3,
+CLOSE,
+MOVE_TO, 6, 2,
+LINE_TO, 7, 2,
+LINE_TO, 7, 3,
+LINE_TO, 6, 3,
+CLOSE,
+MOVE_TO, 9, 2,
+LINE_TO, 10, 2,
+LINE_TO, 10, 3,
+LINE_TO, 9, 3,
+CLOSE,
+MOVE_TO, 13, 2,
+LINE_TO, 14, 2,
+LINE_TO, 14, 3,
+LINE_TO, 13, 3,
+CLOSE,
+MOVE_TO, 1, 3,
+LINE_TO, 2, 3,
+LINE_TO, 2, 4,
+LINE_TO, 1, 4,
+CLOSE,
+MOVE_TO, 6, 3,
+LINE_TO, 7, 3,
+LINE_TO, 7, 4,
+LINE_TO, 6, 4,
+CLOSE,
+MOVE_TO, 9, 3,
+LINE_TO, 10, 3,
+LINE_TO, 10, 4,
+LINE_TO, 9, 4,
+CLOSE,
+MOVE_TO, 13, 3,
+LINE_TO, 14, 3,
+LINE_TO, 14, 4,
+LINE_TO, 13, 4,
+CLOSE,
+MOVE_TO, 1, 4,
+LINE_TO, 2, 4,
+LINE_TO, 2, 5,
+LINE_TO, 1, 5,
+CLOSE,
+MOVE_TO, 6, 4,
+LINE_TO, 7, 4,
+LINE_TO, 7, 5,
+LINE_TO, 6, 5,
+CLOSE,
+MOVE_TO, 9, 4,
+LINE_TO, 10, 4,
+LINE_TO, 10, 5,
+LINE_TO, 9, 5,
+CLOSE,
+MOVE_TO, 13, 4,
+LINE_TO, 14, 4,
+LINE_TO, 14, 5,
+LINE_TO, 13, 5,
+CLOSE,
+MOVE_TO, 1, 5,
+LINE_TO, 7, 5,
+LINE_TO, 7, 6,
+LINE_TO, 1, 6,
+CLOSE,
+MOVE_TO, 9, 5,
+LINE_TO, 14, 5,
+LINE_TO, 14, 6,
+LINE_TO, 9, 6,
+CLOSE,
+MOVE_TO, 1, 7,
+LINE_TO, 7, 7,
+LINE_TO, 7, 8,
+LINE_TO, 1, 8,
+CLOSE,
+MOVE_TO, 8, 7,
+LINE_TO, 14, 7,
+LINE_TO, 14, 8,
+LINE_TO, 8, 8,
+CLOSE,
+MOVE_TO, 1, 8,
+LINE_TO, 2, 8,
+LINE_TO, 2, 9,
+LINE_TO, 1, 9,
+CLOSE,
+MOVE_TO, 6, 8,
+LINE_TO, 7, 8,
+LINE_TO, 7, 9,
+LINE_TO, 6, 9,
+CLOSE,
+MOVE_TO, 8, 8,
+LINE_TO, 9, 8,
+LINE_TO, 9, 9,
+LINE_TO, 8, 9,
+CLOSE,
+MOVE_TO, 13, 8,
+LINE_TO, 14, 8,
+LINE_TO, 14, 9,
+LINE_TO, 13, 9,
+CLOSE,
+MOVE_TO, 1, 9,
+LINE_TO, 2, 9,
+LINE_TO, 2, 10,
+LINE_TO, 1, 10,
+CLOSE,
+MOVE_TO, 6, 9,
+LINE_TO, 7, 9,
+LINE_TO, 7, 10,
+LINE_TO, 6, 10,
+CLOSE,
+MOVE_TO, 8, 9,
+LINE_TO, 9, 9,
+LINE_TO, 9, 10,
+LINE_TO, 8, 10,
+CLOSE,
+MOVE_TO, 13, 9,
+LINE_TO, 14, 9,
+LINE_TO, 14, 10,
+LINE_TO, 13, 10,
+CLOSE,
+MOVE_TO, 1, 10,
+LINE_TO, 2, 10,
+LINE_TO, 2, 11,
+LINE_TO, 1, 11,
+CLOSE,
+MOVE_TO, 6, 10,
+LINE_TO, 7, 10,
+LINE_TO, 7, 11,
+LINE_TO, 6, 11,
+CLOSE,
+MOVE_TO, 8, 10,
+LINE_TO, 9, 10,
+LINE_TO, 9, 11,
+LINE_TO, 8, 11,
+CLOSE,
+MOVE_TO, 13, 10,
+LINE_TO, 14, 10,
+LINE_TO, 14, 11,
+LINE_TO, 13, 11,
+CLOSE,
+MOVE_TO, 1, 11,
+LINE_TO, 2, 11,
+LINE_TO, 2, 12,
+LINE_TO, 1, 12,
+CLOSE,
+MOVE_TO, 6, 11,
+LINE_TO, 7, 11,
+LINE_TO, 7, 12,
+LINE_TO, 6, 12,
+CLOSE,
+MOVE_TO, 8, 11,
+LINE_TO, 9, 11,
+LINE_TO, 9, 12,
+LINE_TO, 8, 12,
+CLOSE,
+MOVE_TO, 13, 11,
+LINE_TO, 14, 11,
+LINE_TO, 14, 12,
+LINE_TO, 13, 12,
+CLOSE,
+MOVE_TO, 1, 12,
+LINE_TO, 7, 12,
+LINE_TO, 7, 13,
+LINE_TO, 1, 13,
+CLOSE,
+MOVE_TO, 8, 12,
+LINE_TO, 14, 12,
+LINE_TO, 14, 13,
+LINE_TO, 8, 13,
+CLOSE,
--- /dev/null
+++ b/chrome/browser/ui/aerographene_theme_icons/pixelized_ext2.icon
@@ -0,0 +1,137 @@
+// Copyright 2026 The aerographene Authors
+// Use of this source code is governed by the GPL-3.0 license.
+// Gerado por theming/design-language/gen_theme_icons.py; não editar.
+
+CANVAS_DIMENSIONS, 16,
+FILL_RULE_NONZERO,
+DISABLE_AA,
+MOVE_TO, 8, 1,
+LINE_TO, 14, 1,
+LINE_TO, 14, 2,
+LINE_TO, 8, 2,
+CLOSE,
+MOVE_TO, 12, 2,
+LINE_TO, 14, 2,
+LINE_TO, 14, 3,
+LINE_TO, 12, 3,
+CLOSE,
+MOVE_TO, 11, 3,
+LINE_TO, 12, 3,
+LINE_TO, 12, 4,
+LINE_TO, 11, 4,
+CLOSE,
+MOVE_TO, 13, 3,
+LINE_TO, 14, 3,
+LINE_TO, 14, 4,
+LINE_TO, 13, 4,
+CLOSE,
+MOVE_TO, 1, 4,
+LINE_TO, 7, 4,
+LINE_TO, 7, 5,
+LINE_TO, 1, 5,
+CLOSE,
+MOVE_TO, 10, 4,
+LINE_TO, 11, 4,
+LINE_TO, 11, 5,
+LINE_TO, 10, 5,
+CLOSE,
+MOVE_TO, 13, 4,
+LINE_TO, 14, 4,
+LINE_TO, 14, 5,
+LINE_TO, 13, 5,
+CLOSE,
+MOVE_TO, 1, 5,
+LINE_TO, 2, 5,
+LINE_TO, 2, 6,
+LINE_TO, 1, 6,
+CLOSE,
+MOVE_TO, 9, 5,
+LINE_TO, 10, 5,
+LINE_TO, 10, 6,
+LINE_TO, 9, 6,
+CLOSE,
+MOVE_TO, 13, 5,
+LINE_TO, 14, 5,
+LINE_TO, 14, 6,
+LINE_TO, 13, 6,
+CLOSE,
+MOVE_TO, 1, 6,
+LINE_TO, 2, 6,
+LINE_TO, 2, 7,
+LINE_TO, 1, 7,
+CLOSE,
+MOVE_TO, 8, 6,
+LINE_TO, 9, 6,
+LINE_TO, 9, 7,
+LINE_TO, 8, 7,
+CLOSE,
+MOVE_TO, 13, 6,
+LINE_TO, 14, 6,
+LINE_TO, 14, 7,
+LINE_TO, 13, 7,
+CLOSE,
+MOVE_TO, 1, 7,
+LINE_TO, 2, 7,
+LINE_TO, 2, 8,
+LINE_TO, 1, 8,
+CLOSE,
+MOVE_TO, 7, 7,
+LINE_TO, 8, 7,
+LINE_TO, 8, 8,
+LINE_TO, 7, 8,
+CLOSE,
+MOVE_TO, 1, 8,
+LINE_TO, 2, 8,
+LINE_TO, 2, 9,
+LINE_TO, 1, 9,
+CLOSE,
+MOVE_TO, 6, 8,
+LINE_TO, 7, 8,
+LINE_TO, 7, 9,
+LINE_TO, 6, 9,
+CLOSE,
+MOVE_TO, 1, 9,
+LINE_TO, 2, 9,
+LINE_TO, 2, 10,
+LINE_TO, 1, 10,
+CLOSE,
+MOVE_TO, 10, 9,
+LINE_TO, 11, 9,
+LINE_TO, 11, 10,
+LINE_TO, 10, 10,
+CLOSE,
+MOVE_TO, 1, 10,
+LINE_TO, 2, 10,
+LINE_TO, 2, 11,
+LINE_TO, 1, 11,
+CLOSE,
+MOVE_TO, 10, 10,
+LINE_TO, 11, 10,
+LINE_TO, 11, 11,
+LINE_TO, 10, 11,
+CLOSE,
+MOVE_TO, 1, 11,
+LINE_TO, 2, 11,
+LINE_TO, 2, 12,
+LINE_TO, 1, 12,
+CLOSE,
+MOVE_TO, 10, 11,
+LINE_TO, 11, 11,
+LINE_TO, 11, 12,
+LINE_TO, 10, 12,
+CLOSE,
+MOVE_TO, 1, 12,
+LINE_TO, 2, 12,
+LINE_TO, 2, 13,
+LINE_TO, 1, 13,
+CLOSE,
+MOVE_TO, 10, 12,
+LINE_TO, 11, 12,
+LINE_TO, 11, 13,
+LINE_TO, 10, 13,
+CLOSE,
+MOVE_TO, 1, 13,
+LINE_TO, 11, 13,
+LINE_TO, 11, 14,
+LINE_TO, 1, 14,
+CLOSE,
--- /dev/null
+++ b/chrome/browser/ui/aerographene_theme_icons/pixelized_folder.icon
@@ -0,0 +1,210 @@
+// Copyright 2026 The aerographene Authors
+// Use of this source code is governed by the GPL-3.0 license.
+// Gerado por theming/design-language/gen_theme_icons.py; não editar.
+
+CANVAS_DIMENSIONS, 16,
+PATH_COLOR_ALPHA, 0x38,
+FILL_RULE_NONZERO,
+DISABLE_AA,
+MOVE_TO, 2, 3,
+LINE_TO, 7, 3,
+LINE_TO, 7, 4,
+LINE_TO, 2, 4,
+CLOSE,
+MOVE_TO, 2, 4,
+LINE_TO, 13, 4,
+LINE_TO, 13, 5,
+LINE_TO, 2, 5,
+CLOSE,
+MOVE_TO, 2, 5,
+LINE_TO, 13, 5,
+LINE_TO, 13, 6,
+LINE_TO, 2, 6,
+CLOSE,
+MOVE_TO, 2, 6,
+LINE_TO, 13, 6,
+LINE_TO, 13, 7,
+LINE_TO, 2, 7,
+CLOSE,
+MOVE_TO, 2, 7,
+LINE_TO, 13, 7,
+LINE_TO, 13, 8,
+LINE_TO, 2, 8,
+CLOSE,
+MOVE_TO, 2, 8,
+LINE_TO, 13, 8,
+LINE_TO, 13, 9,
+LINE_TO, 2, 9,
+CLOSE,
+MOVE_TO, 2, 9,
+LINE_TO, 13, 9,
+LINE_TO, 13, 10,
+LINE_TO, 2, 10,
+CLOSE,
+MOVE_TO, 2, 10,
+LINE_TO, 13, 10,
+LINE_TO, 13, 11,
+LINE_TO, 2, 11,
+CLOSE,
+MOVE_TO, 2, 11,
+LINE_TO, 13, 11,
+LINE_TO, 13, 12,
+LINE_TO, 2, 12,
+CLOSE,
+NEW_PATH,
+PATH_COLOR_ALPHA, 0x80,
+FILL_RULE_NONZERO,
+DISABLE_AA,
+MOVE_TO, 14, 4,
+LINE_TO, 15, 4,
+LINE_TO, 15, 5,
+LINE_TO, 14, 5,
+CLOSE,
+MOVE_TO, 14, 5,
+LINE_TO, 15, 5,
+LINE_TO, 15, 6,
+LINE_TO, 14, 6,
+CLOSE,
+MOVE_TO, 14, 6,
+LINE_TO, 15, 6,
+LINE_TO, 15, 7,
+LINE_TO, 14, 7,
+CLOSE,
+MOVE_TO, 14, 7,
+LINE_TO, 15, 7,
+LINE_TO, 15, 8,
+LINE_TO, 14, 8,
+CLOSE,
+MOVE_TO, 14, 8,
+LINE_TO, 15, 8,
+LINE_TO, 15, 9,
+LINE_TO, 14, 9,
+CLOSE,
+MOVE_TO, 14, 9,
+LINE_TO, 15, 9,
+LINE_TO, 15, 10,
+LINE_TO, 14, 10,
+CLOSE,
+MOVE_TO, 14, 10,
+LINE_TO, 15, 10,
+LINE_TO, 15, 11,
+LINE_TO, 14, 11,
+CLOSE,
+MOVE_TO, 14, 11,
+LINE_TO, 15, 11,
+LINE_TO, 15, 12,
+LINE_TO, 14, 12,
+CLOSE,
+MOVE_TO, 2, 13,
+LINE_TO, 13, 13,
+LINE_TO, 13, 14,
+LINE_TO, 2, 14,
+CLOSE,
+NEW_PATH,
+FILL_RULE_NONZERO,
+DISABLE_AA,
+MOVE_TO, 1, 1,
+LINE_TO, 6, 1,
+LINE_TO, 6, 2,
+LINE_TO, 1, 2,
+CLOSE,
+MOVE_TO, 1, 2,
+LINE_TO, 7, 2,
+LINE_TO, 7, 3,
+LINE_TO, 1, 3,
+CLOSE,
+MOVE_TO, 1, 3,
+LINE_TO, 2, 3,
+LINE_TO, 2, 4,
+LINE_TO, 1, 4,
+CLOSE,
+MOVE_TO, 7, 3,
+LINE_TO, 14, 3,
+LINE_TO, 14, 4,
+LINE_TO, 7, 4,
+CLOSE,
+MOVE_TO, 1, 4,
+LINE_TO, 2, 4,
+LINE_TO, 2, 5,
+LINE_TO, 1, 5,
+CLOSE,
+MOVE_TO, 13, 4,
+LINE_TO, 14, 4,
+LINE_TO, 14, 5,
+LINE_TO, 13, 5,
+CLOSE,
+MOVE_TO, 1, 5,
+LINE_TO, 2, 5,
+LINE_TO, 2, 6,
+LINE_TO, 1, 6,
+CLOSE,
+MOVE_TO, 13, 5,
+LINE_TO, 14, 5,
+LINE_TO, 14, 6,
+LINE_TO, 13, 6,
+CLOSE,
+MOVE_TO, 1, 6,
+LINE_TO, 2, 6,
+LINE_TO, 2, 7,
+LINE_TO, 1, 7,
+CLOSE,
+MOVE_TO, 13, 6,
+LINE_TO, 14, 6,
+LINE_TO, 14, 7,
+LINE_TO, 13, 7,
+CLOSE,
+MOVE_TO, 1, 7,
+LINE_TO, 2, 7,
+LINE_TO, 2, 8,
+LINE_TO, 1, 8,
+CLOSE,
+MOVE_TO, 13, 7,
+LINE_TO, 14, 7,
+LINE_TO, 14, 8,
+LINE_TO, 13, 8,
+CLOSE,
+MOVE_TO, 1, 8,
+LINE_TO, 2, 8,
+LINE_TO, 2, 9,
+LINE_TO, 1, 9,
+CLOSE,
+MOVE_TO, 13, 8,
+LINE_TO, 14, 8,
+LINE_TO, 14, 9,
+LINE_TO, 13, 9,
+CLOSE,
+MOVE_TO, 1, 9,
+LINE_TO, 2, 9,
+LINE_TO, 2, 10,
+LINE_TO, 1, 10,
+CLOSE,
+MOVE_TO, 13, 9,
+LINE_TO, 14, 9,
+LINE_TO, 14, 10,
+LINE_TO, 13, 10,
+CLOSE,
+MOVE_TO, 1, 10,
+LINE_TO, 2, 10,
+LINE_TO, 2, 11,
+LINE_TO, 1, 11,
+CLOSE,
+MOVE_TO, 13, 10,
+LINE_TO, 14, 10,
+LINE_TO, 14, 11,
+LINE_TO, 13, 11,
+CLOSE,
+MOVE_TO, 1, 11,
+LINE_TO, 2, 11,
+LINE_TO, 2, 12,
+LINE_TO, 1, 12,
+CLOSE,
+MOVE_TO, 13, 11,
+LINE_TO, 14, 11,
+LINE_TO, 14, 12,
+LINE_TO, 13, 12,
+CLOSE,
+MOVE_TO, 1, 12,
+LINE_TO, 14, 12,
+LINE_TO, 14, 13,
+LINE_TO, 1, 13,
+CLOSE,
--- /dev/null
+++ b/chrome/browser/ui/aerographene_theme_icons/pixelized_fwd.icon
@@ -0,0 +1,62 @@
+// Copyright 2026 The aerographene Authors
+// Use of this source code is governed by the GPL-3.0 license.
+// Gerado por theming/design-language/gen_theme_icons.py; não editar.
+
+CANVAS_DIMENSIONS, 16,
+FILL_RULE_NONZERO,
+DISABLE_AA,
+MOVE_TO, 8, 2,
+LINE_TO, 9, 2,
+LINE_TO, 9, 3,
+LINE_TO, 8, 3,
+CLOSE,
+MOVE_TO, 9, 3,
+LINE_TO, 10, 3,
+LINE_TO, 10, 4,
+LINE_TO, 9, 4,
+CLOSE,
+MOVE_TO, 10, 4,
+LINE_TO, 11, 4,
+LINE_TO, 11, 5,
+LINE_TO, 10, 5,
+CLOSE,
+MOVE_TO, 11, 5,
+LINE_TO, 12, 5,
+LINE_TO, 12, 6,
+LINE_TO, 11, 6,
+CLOSE,
+MOVE_TO, 12, 6,
+LINE_TO, 13, 6,
+LINE_TO, 13, 7,
+LINE_TO, 12, 7,
+CLOSE,
+MOVE_TO, 1, 7,
+LINE_TO, 14, 7,
+LINE_TO, 14, 8,
+LINE_TO, 1, 8,
+CLOSE,
+MOVE_TO, 12, 8,
+LINE_TO, 13, 8,
+LINE_TO, 13, 9,
+LINE_TO, 12, 9,
+CLOSE,
+MOVE_TO, 11, 9,
+LINE_TO, 12, 9,
+LINE_TO, 12, 10,
+LINE_TO, 11, 10,
+CLOSE,
+MOVE_TO, 10, 10,
+LINE_TO, 11, 10,
+LINE_TO, 11, 11,
+LINE_TO, 10, 11,
+CLOSE,
+MOVE_TO, 9, 11,
+LINE_TO, 10, 11,
+LINE_TO, 10, 12,
+LINE_TO, 9, 12,
+CLOSE,
+MOVE_TO, 8, 12,
+LINE_TO, 9, 12,
+LINE_TO, 9, 13,
+LINE_TO, 8, 13,
+CLOSE,
--- /dev/null
+++ b/chrome/browser/ui/aerographene_theme_icons/pixelized_gear.icon
@@ -0,0 +1,360 @@
+// Copyright 2026 The aerographene Authors
+// Use of this source code is governed by the GPL-3.0 license.
+// Gerado por theming/design-language/gen_theme_icons.py; não editar.
+
+CANVAS_DIMENSIONS, 16,
+PATH_COLOR_ALPHA, 0x38,
+FILL_RULE_NONZERO,
+DISABLE_AA,
+MOVE_TO, 7, 2,
+LINE_TO, 8, 2,
+LINE_TO, 8, 3,
+LINE_TO, 7, 3,
+CLOSE,
+MOVE_TO, 7, 3,
+LINE_TO, 8, 3,
+LINE_TO, 8, 4,
+LINE_TO, 7, 4,
+CLOSE,
+MOVE_TO, 3, 4,
+LINE_TO, 4, 4,
+LINE_TO, 4, 5,
+LINE_TO, 3, 5,
+CLOSE,
+MOVE_TO, 7, 4,
+LINE_TO, 8, 4,
+LINE_TO, 8, 5,
+LINE_TO, 7, 5,
+CLOSE,
+MOVE_TO, 11, 4,
+LINE_TO, 12, 4,
+LINE_TO, 12, 5,
+LINE_TO, 11, 5,
+CLOSE,
+MOVE_TO, 4, 5,
+LINE_TO, 11, 5,
+LINE_TO, 11, 6,
+LINE_TO, 4, 6,
+CLOSE,
+MOVE_TO, 4, 6,
+LINE_TO, 6, 6,
+LINE_TO, 6, 7,
+LINE_TO, 4, 7,
+CLOSE,
+MOVE_TO, 9, 6,
+LINE_TO, 11, 6,
+LINE_TO, 11, 7,
+LINE_TO, 9, 7,
+CLOSE,
+MOVE_TO, 2, 7,
+LINE_TO, 5, 7,
+LINE_TO, 5, 8,
+LINE_TO, 2, 8,
+CLOSE,
+MOVE_TO, 10, 7,
+LINE_TO, 13, 7,
+LINE_TO, 13, 8,
+LINE_TO, 10, 8,
+CLOSE,
+MOVE_TO, 4, 8,
+LINE_TO, 6, 8,
+LINE_TO, 6, 9,
+LINE_TO, 4, 9,
+CLOSE,
+MOVE_TO, 9, 8,
+LINE_TO, 11, 8,
+LINE_TO, 11, 9,
+LINE_TO, 9, 9,
+CLOSE,
+MOVE_TO, 4, 9,
+LINE_TO, 11, 9,
+LINE_TO, 11, 10,
+LINE_TO, 4, 10,
+CLOSE,
+MOVE_TO, 3, 10,
+LINE_TO, 4, 10,
+LINE_TO, 4, 11,
+LINE_TO, 3, 11,
+CLOSE,
+MOVE_TO, 7, 10,
+LINE_TO, 8, 10,
+LINE_TO, 8, 11,
+LINE_TO, 7, 11,
+CLOSE,
+MOVE_TO, 11, 10,
+LINE_TO, 12, 10,
+LINE_TO, 12, 11,
+LINE_TO, 11, 11,
+CLOSE,
+MOVE_TO, 7, 11,
+LINE_TO, 8, 11,
+LINE_TO, 8, 12,
+LINE_TO, 7, 12,
+CLOSE,
+MOVE_TO, 7, 12,
+LINE_TO, 8, 12,
+LINE_TO, 8, 13,
+LINE_TO, 7, 13,
+CLOSE,
+NEW_PATH,
+PATH_COLOR_ALPHA, 0x80,
+FILL_RULE_NONZERO,
+DISABLE_AA,
+MOVE_TO, 9, 2,
+LINE_TO, 10, 2,
+LINE_TO, 10, 3,
+LINE_TO, 9, 3,
+CLOSE,
+MOVE_TO, 9, 3,
+LINE_TO, 10, 3,
+LINE_TO, 10, 4,
+LINE_TO, 9, 4,
+CLOSE,
+MOVE_TO, 13, 4,
+LINE_TO, 14, 4,
+LINE_TO, 14, 5,
+LINE_TO, 13, 5,
+CLOSE,
+MOVE_TO, 12, 5,
+LINE_TO, 13, 5,
+LINE_TO, 13, 6,
+LINE_TO, 12, 6,
+CLOSE,
+MOVE_TO, 6, 7,
+LINE_TO, 9, 7,
+LINE_TO, 9, 8,
+LINE_TO, 6, 8,
+CLOSE,
+MOVE_TO, 14, 7,
+LINE_TO, 15, 7,
+LINE_TO, 15, 8,
+LINE_TO, 14, 8,
+CLOSE,
+MOVE_TO, 2, 9,
+LINE_TO, 3, 9,
+LINE_TO, 3, 10,
+LINE_TO, 2, 10,
+CLOSE,
+MOVE_TO, 12, 9,
+LINE_TO, 13, 9,
+LINE_TO, 13, 10,
+LINE_TO, 12, 10,
+CLOSE,
+MOVE_TO, 13, 10,
+LINE_TO, 14, 10,
+LINE_TO, 14, 11,
+LINE_TO, 13, 11,
+CLOSE,
+MOVE_TO, 4, 11,
+LINE_TO, 6, 11,
+LINE_TO, 6, 12,
+LINE_TO, 4, 12,
+CLOSE,
+MOVE_TO, 9, 11,
+LINE_TO, 11, 11,
+LINE_TO, 11, 12,
+LINE_TO, 9, 12,
+CLOSE,
+MOVE_TO, 3, 12,
+LINE_TO, 4, 12,
+LINE_TO, 4, 13,
+LINE_TO, 3, 13,
+CLOSE,
+MOVE_TO, 9, 12,
+LINE_TO, 10, 12,
+LINE_TO, 10, 13,
+LINE_TO, 9, 13,
+CLOSE,
+MOVE_TO, 11, 12,
+LINE_TO, 12, 12,
+LINE_TO, 12, 13,
+LINE_TO, 11, 13,
+CLOSE,
+MOVE_TO, 7, 14,
+LINE_TO, 8, 14,
+LINE_TO, 8, 15,
+LINE_TO, 7, 15,
+CLOSE,
+NEW_PATH,
+FILL_RULE_NONZERO,
+DISABLE_AA,
+MOVE_TO, 6, 1,
+LINE_TO, 9, 1,
+LINE_TO, 9, 2,
+LINE_TO, 6, 2,
+CLOSE,
+MOVE_TO, 6, 2,
+LINE_TO, 7, 2,
+LINE_TO, 7, 3,
+LINE_TO, 6, 3,
+CLOSE,
+MOVE_TO, 8, 2,
+LINE_TO, 9, 2,
+LINE_TO, 9, 3,
+LINE_TO, 8, 3,
+CLOSE,
+MOVE_TO, 2, 3,
+LINE_TO, 4, 3,
+LINE_TO, 4, 4,
+LINE_TO, 2, 4,
+CLOSE,
+MOVE_TO, 6, 3,
+LINE_TO, 7, 3,
+LINE_TO, 7, 4,
+LINE_TO, 6, 4,
+CLOSE,
+MOVE_TO, 8, 3,
+LINE_TO, 9, 3,
+LINE_TO, 9, 4,
+LINE_TO, 8, 4,
+CLOSE,
+MOVE_TO, 11, 3,
+LINE_TO, 13, 3,
+LINE_TO, 13, 4,
+LINE_TO, 11, 4,
+CLOSE,
+MOVE_TO, 2, 4,
+LINE_TO, 3, 4,
+LINE_TO, 3, 5,
+LINE_TO, 2, 5,
+CLOSE,
+MOVE_TO, 4, 4,
+LINE_TO, 7, 4,
+LINE_TO, 7, 5,
+LINE_TO, 4, 5,
+CLOSE,
+MOVE_TO, 8, 4,
+LINE_TO, 11, 4,
+LINE_TO, 11, 5,
+LINE_TO, 8, 5,
+CLOSE,
+MOVE_TO, 12, 4,
+LINE_TO, 13, 4,
+LINE_TO, 13, 5,
+LINE_TO, 12, 5,
+CLOSE,
+MOVE_TO, 3, 5,
+LINE_TO, 4, 5,
+LINE_TO, 4, 6,
+LINE_TO, 3, 6,
+CLOSE,
+MOVE_TO, 11, 5,
+LINE_TO, 12, 5,
+LINE_TO, 12, 6,
+LINE_TO, 11, 6,
+CLOSE,
+MOVE_TO, 1, 6,
+LINE_TO, 4, 6,
+LINE_TO, 4, 7,
+LINE_TO, 1, 7,
+CLOSE,
+MOVE_TO, 6, 6,
+LINE_TO, 9, 6,
+LINE_TO, 9, 7,
+LINE_TO, 6, 7,
+CLOSE,
+MOVE_TO, 11, 6,
+LINE_TO, 14, 6,
+LINE_TO, 14, 7,
+LINE_TO, 11, 7,
+CLOSE,
+MOVE_TO, 1, 7,
+LINE_TO, 2, 7,
+LINE_TO, 2, 8,
+LINE_TO, 1, 8,
+CLOSE,
+MOVE_TO, 5, 7,
+LINE_TO, 6, 7,
+LINE_TO, 6, 8,
+LINE_TO, 5, 8,
+CLOSE,
+MOVE_TO, 9, 7,
+LINE_TO, 10, 7,
+LINE_TO, 10, 8,
+LINE_TO, 9, 8,
+CLOSE,
+MOVE_TO, 13, 7,
+LINE_TO, 14, 7,
+LINE_TO, 14, 8,
+LINE_TO, 13, 8,
+CLOSE,
+MOVE_TO, 1, 8,
+LINE_TO, 4, 8,
+LINE_TO, 4, 9,
+LINE_TO, 1, 9,
+CLOSE,
+MOVE_TO, 6, 8,
+LINE_TO, 9, 8,
+LINE_TO, 9, 9,
+LINE_TO, 6, 9,
+CLOSE,
+MOVE_TO, 11, 8,
+LINE_TO, 14, 8,
+LINE_TO, 14, 9,
+LINE_TO, 11, 9,
+CLOSE,
+MOVE_TO, 3, 9,
+LINE_TO, 4, 9,
+LINE_TO, 4, 10,
+LINE_TO, 3, 10,
+CLOSE,
+MOVE_TO, 11, 9,
+LINE_TO, 12, 9,
+LINE_TO, 12, 10,
+LINE_TO, 11, 10,
+CLOSE,
+MOVE_TO, 2, 10,
+LINE_TO, 3, 10,
+LINE_TO, 3, 11,
+LINE_TO, 2, 11,
+CLOSE,
+MOVE_TO, 4, 10,
+LINE_TO, 7, 10,
+LINE_TO, 7, 11,
+LINE_TO, 4, 11,
+CLOSE,
+MOVE_TO, 8, 10,
+LINE_TO, 11, 10,
+LINE_TO, 11, 11,
+LINE_TO, 8, 11,
+CLOSE,
+MOVE_TO, 12, 10,
+LINE_TO, 13, 10,
+LINE_TO, 13, 11,
+LINE_TO, 12, 11,
+CLOSE,
+MOVE_TO, 2, 11,
+LINE_TO, 4, 11,
+LINE_TO, 4, 12,
+LINE_TO, 2, 12,
+CLOSE,
+MOVE_TO, 6, 11,
+LINE_TO, 7, 11,
+LINE_TO, 7, 12,
+LINE_TO, 6, 12,
+CLOSE,
+MOVE_TO, 8, 11,
+LINE_TO, 9, 11,
+LINE_TO, 9, 12,
+LINE_TO, 8, 12,
+CLOSE,
+MOVE_TO, 11, 11,
+LINE_TO, 13, 11,
+LINE_TO, 13, 12,
+LINE_TO, 11, 12,
+CLOSE,
+MOVE_TO, 6, 12,
+LINE_TO, 7, 12,
+LINE_TO, 7, 13,
+LINE_TO, 6, 13,
+CLOSE,
+MOVE_TO, 8, 12,
+LINE_TO, 9, 12,
+LINE_TO, 9, 13,
+LINE_TO, 8, 13,
+CLOSE,
+MOVE_TO, 6, 13,
+LINE_TO, 9, 13,
+LINE_TO, 9, 14,
+LINE_TO, 6, 14,
+CLOSE,
--- /dev/null
+++ b/chrome/browser/ui/aerographene_theme_icons/pixelized_hist.icon
@@ -0,0 +1,290 @@
+// Copyright 2026 The aerographene Authors
+// Use of this source code is governed by the GPL-3.0 license.
+// Gerado por theming/design-language/gen_theme_icons.py; não editar.
+
+CANVAS_DIMENSIONS, 16,
+PATH_COLOR_ALPHA, 0x38,
+FILL_RULE_NONZERO,
+DISABLE_AA,
+MOVE_TO, 5, 2,
+LINE_TO, 10, 2,
+LINE_TO, 10, 3,
+LINE_TO, 5, 3,
+CLOSE,
+MOVE_TO, 3, 3,
+LINE_TO, 12, 3,
+LINE_TO, 12, 4,
+LINE_TO, 3, 4,
+CLOSE,
+MOVE_TO, 3, 4,
+LINE_TO, 7, 4,
+LINE_TO, 7, 5,
+LINE_TO, 3, 5,
+CLOSE,
+MOVE_TO, 8, 4,
+LINE_TO, 12, 4,
+LINE_TO, 12, 5,
+LINE_TO, 8, 5,
+CLOSE,
+MOVE_TO, 2, 5,
+LINE_TO, 7, 5,
+LINE_TO, 7, 6,
+LINE_TO, 2, 6,
+CLOSE,
+MOVE_TO, 8, 5,
+LINE_TO, 13, 5,
+LINE_TO, 13, 6,
+LINE_TO, 8, 6,
+CLOSE,
+MOVE_TO, 2, 6,
+LINE_TO, 7, 6,
+LINE_TO, 7, 7,
+LINE_TO, 2, 7,
+CLOSE,
+MOVE_TO, 8, 6,
+LINE_TO, 13, 6,
+LINE_TO, 13, 7,
+LINE_TO, 8, 7,
+CLOSE,
+MOVE_TO, 2, 7,
+LINE_TO, 7, 7,
+LINE_TO, 7, 8,
+LINE_TO, 2, 8,
+CLOSE,
+MOVE_TO, 11, 7,
+LINE_TO, 13, 7,
+LINE_TO, 13, 8,
+LINE_TO, 11, 8,
+CLOSE,
+MOVE_TO, 2, 8,
+LINE_TO, 13, 8,
+LINE_TO, 13, 9,
+LINE_TO, 2, 9,
+CLOSE,
+MOVE_TO, 2, 9,
+LINE_TO, 13, 9,
+LINE_TO, 13, 10,
+LINE_TO, 2, 10,
+CLOSE,
+MOVE_TO, 3, 10,
+LINE_TO, 12, 10,
+LINE_TO, 12, 11,
+LINE_TO, 3, 11,
+CLOSE,
+MOVE_TO, 3, 11,
+LINE_TO, 12, 11,
+LINE_TO, 12, 12,
+LINE_TO, 3, 12,
+CLOSE,
+MOVE_TO, 5, 12,
+LINE_TO, 10, 12,
+LINE_TO, 10, 13,
+LINE_TO, 5, 13,
+CLOSE,
+NEW_PATH,
+PATH_COLOR_ALPHA, 0x80,
+FILL_RULE_NONZERO,
+DISABLE_AA,
+MOVE_TO, 13, 3,
+LINE_TO, 14, 3,
+LINE_TO, 14, 4,
+LINE_TO, 13, 4,
+CLOSE,
+MOVE_TO, 13, 4,
+LINE_TO, 14, 4,
+LINE_TO, 14, 5,
+LINE_TO, 13, 5,
+CLOSE,
+MOVE_TO, 14, 5,
+LINE_TO, 15, 5,
+LINE_TO, 15, 6,
+LINE_TO, 14, 6,
+CLOSE,
+MOVE_TO, 14, 6,
+LINE_TO, 15, 6,
+LINE_TO, 15, 7,
+LINE_TO, 14, 7,
+CLOSE,
+MOVE_TO, 14, 7,
+LINE_TO, 15, 7,
+LINE_TO, 15, 8,
+LINE_TO, 14, 8,
+CLOSE,
+MOVE_TO, 14, 8,
+LINE_TO, 15, 8,
+LINE_TO, 15, 9,
+LINE_TO, 14, 9,
+CLOSE,
+MOVE_TO, 14, 9,
+LINE_TO, 15, 9,
+LINE_TO, 15, 10,
+LINE_TO, 14, 10,
+CLOSE,
+MOVE_TO, 13, 10,
+LINE_TO, 15, 10,
+LINE_TO, 15, 11,
+LINE_TO, 13, 11,
+CLOSE,
+MOVE_TO, 13, 11,
+LINE_TO, 14, 11,
+LINE_TO, 14, 12,
+LINE_TO, 13, 12,
+CLOSE,
+MOVE_TO, 3, 13,
+LINE_TO, 5, 13,
+LINE_TO, 5, 14,
+LINE_TO, 3, 14,
+CLOSE,
+MOVE_TO, 10, 13,
+LINE_TO, 12, 13,
+LINE_TO, 12, 14,
+LINE_TO, 10, 14,
+CLOSE,
+MOVE_TO, 5, 14,
+LINE_TO, 10, 14,
+LINE_TO, 10, 15,
+LINE_TO, 5, 15,
+CLOSE,
+NEW_PATH,
+FILL_RULE_NONZERO,
+DISABLE_AA,
+MOVE_TO, 5, 1,
+LINE_TO, 10, 1,
+LINE_TO, 10, 2,
+LINE_TO, 5, 2,
+CLOSE,
+MOVE_TO, 3, 2,
+LINE_TO, 5, 2,
+LINE_TO, 5, 3,
+LINE_TO, 3, 3,
+CLOSE,
+MOVE_TO, 10, 2,
+LINE_TO, 12, 2,
+LINE_TO, 12, 3,
+LINE_TO, 10, 3,
+CLOSE,
+MOVE_TO, 2, 3,
+LINE_TO, 3, 3,
+LINE_TO, 3, 4,
+LINE_TO, 2, 4,
+CLOSE,
+MOVE_TO, 12, 3,
+LINE_TO, 13, 3,
+LINE_TO, 13, 4,
+LINE_TO, 12, 4,
+CLOSE,
+MOVE_TO, 2, 4,
+LINE_TO, 3, 4,
+LINE_TO, 3, 5,
+LINE_TO, 2, 5,
+CLOSE,
+MOVE_TO, 7, 4,
+LINE_TO, 8, 4,
+LINE_TO, 8, 5,
+LINE_TO, 7, 5,
+CLOSE,
+MOVE_TO, 12, 4,
+LINE_TO, 13, 4,
+LINE_TO, 13, 5,
+LINE_TO, 12, 5,
+CLOSE,
+MOVE_TO, 1, 5,
+LINE_TO, 2, 5,
+LINE_TO, 2, 6,
+LINE_TO, 1, 6,
+CLOSE,
+MOVE_TO, 7, 5,
+LINE_TO, 8, 5,
+LINE_TO, 8, 6,
+LINE_TO, 7, 6,
+CLOSE,
+MOVE_TO, 13, 5,
+LINE_TO, 14, 5,
+LINE_TO, 14, 6,
+LINE_TO, 13, 6,
+CLOSE,
+MOVE_TO, 1, 6,
+LINE_TO, 2, 6,
+LINE_TO, 2, 7,
+LINE_TO, 1, 7,
+CLOSE,
+MOVE_TO, 7, 6,
+LINE_TO, 8, 6,
+LINE_TO, 8, 7,
+LINE_TO, 7, 7,
+CLOSE,
+MOVE_TO, 13, 6,
+LINE_TO, 14, 6,
+LINE_TO, 14, 7,
+LINE_TO, 13, 7,
+CLOSE,
+MOVE_TO, 1, 7,
+LINE_TO, 2, 7,
+LINE_TO, 2, 8,
+LINE_TO, 1, 8,
+CLOSE,
+MOVE_TO, 7, 7,
+LINE_TO, 11, 7,
+LINE_TO, 11, 8,
+LINE_TO, 7, 8,
+CLOSE,
+MOVE_TO, 13, 7,
+LINE_TO, 14, 7,
+LINE_TO, 14, 8,
+LINE_TO, 13, 8,
+CLOSE,
+MOVE_TO, 1, 8,
+LINE_TO, 2, 8,
+LINE_TO, 2, 9,
+LINE_TO, 1, 9,
+CLOSE,
+MOVE_TO, 13, 8,
+LINE_TO, 14, 8,
+LINE_TO, 14, 9,
+LINE_TO, 13, 9,
+CLOSE,
+MOVE_TO, 1, 9,
+LINE_TO, 2, 9,
+LINE_TO, 2, 10,
+LINE_TO, 1, 10,
+CLOSE,
+MOVE_TO, 13, 9,
+LINE_TO, 14, 9,
+LINE_TO, 14, 10,
+LINE_TO, 13, 10,
+CLOSE,
+MOVE_TO, 2, 10,
+LINE_TO, 3, 10,
+LINE_TO, 3, 11,
+LINE_TO, 2, 11,
+CLOSE,
+MOVE_TO, 12, 10,
+LINE_TO, 13, 10,
+LINE_TO, 13, 11,
+LINE_TO, 12, 11,
+CLOSE,
+MOVE_TO, 2, 11,
+LINE_TO, 3, 11,
+LINE_TO, 3, 12,
+LINE_TO, 2, 12,
+CLOSE,
+MOVE_TO, 12, 11,
+LINE_TO, 13, 11,
+LINE_TO, 13, 12,
+LINE_TO, 12, 12,
+CLOSE,
+MOVE_TO, 3, 12,
+LINE_TO, 5, 12,
+LINE_TO, 5, 13,
+LINE_TO, 3, 13,
+CLOSE,
+MOVE_TO, 10, 12,
+LINE_TO, 12, 12,
+LINE_TO, 12, 13,
+LINE_TO, 10, 13,
+CLOSE,
+MOVE_TO, 5, 13,
+LINE_TO, 10, 13,
+LINE_TO, 10, 14,
+LINE_TO, 5, 14,
+CLOSE,
--- /dev/null
+++ b/chrome/browser/ui/aerographene_theme_icons/pixelized_info.icon
@@ -0,0 +1,310 @@
+// Copyright 2026 The aerographene Authors
+// Use of this source code is governed by the GPL-3.0 license.
+// Gerado por theming/design-language/gen_theme_icons.py; não editar.
+
+CANVAS_DIMENSIONS, 16,
+PATH_COLOR_ALPHA, 0x38,
+FILL_RULE_NONZERO,
+DISABLE_AA,
+MOVE_TO, 5, 2,
+LINE_TO, 10, 2,
+LINE_TO, 10, 3,
+LINE_TO, 5, 3,
+CLOSE,
+MOVE_TO, 3, 3,
+LINE_TO, 12, 3,
+LINE_TO, 12, 4,
+LINE_TO, 3, 4,
+CLOSE,
+MOVE_TO, 3, 4,
+LINE_TO, 7, 4,
+LINE_TO, 7, 5,
+LINE_TO, 3, 5,
+CLOSE,
+MOVE_TO, 8, 4,
+LINE_TO, 12, 4,
+LINE_TO, 12, 5,
+LINE_TO, 8, 5,
+CLOSE,
+MOVE_TO, 2, 5,
+LINE_TO, 13, 5,
+LINE_TO, 13, 6,
+LINE_TO, 2, 6,
+CLOSE,
+MOVE_TO, 2, 6,
+LINE_TO, 7, 6,
+LINE_TO, 7, 7,
+LINE_TO, 2, 7,
+CLOSE,
+MOVE_TO, 8, 6,
+LINE_TO, 13, 6,
+LINE_TO, 13, 7,
+LINE_TO, 8, 7,
+CLOSE,
+MOVE_TO, 2, 7,
+LINE_TO, 7, 7,
+LINE_TO, 7, 8,
+LINE_TO, 2, 8,
+CLOSE,
+MOVE_TO, 8, 7,
+LINE_TO, 13, 7,
+LINE_TO, 13, 8,
+LINE_TO, 8, 8,
+CLOSE,
+MOVE_TO, 2, 8,
+LINE_TO, 7, 8,
+LINE_TO, 7, 9,
+LINE_TO, 2, 9,
+CLOSE,
+MOVE_TO, 8, 8,
+LINE_TO, 13, 8,
+LINE_TO, 13, 9,
+LINE_TO, 8, 9,
+CLOSE,
+MOVE_TO, 2, 9,
+LINE_TO, 7, 9,
+LINE_TO, 7, 10,
+LINE_TO, 2, 10,
+CLOSE,
+MOVE_TO, 8, 9,
+LINE_TO, 13, 9,
+LINE_TO, 13, 10,
+LINE_TO, 8, 10,
+CLOSE,
+MOVE_TO, 3, 10,
+LINE_TO, 7, 10,
+LINE_TO, 7, 11,
+LINE_TO, 3, 11,
+CLOSE,
+MOVE_TO, 8, 10,
+LINE_TO, 12, 10,
+LINE_TO, 12, 11,
+LINE_TO, 8, 11,
+CLOSE,
+MOVE_TO, 3, 11,
+LINE_TO, 12, 11,
+LINE_TO, 12, 12,
+LINE_TO, 3, 12,
+CLOSE,
+MOVE_TO, 5, 12,
+LINE_TO, 10, 12,
+LINE_TO, 10, 13,
+LINE_TO, 5, 13,
+CLOSE,
+NEW_PATH,
+PATH_COLOR_ALPHA, 0x80,
+FILL_RULE_NONZERO,
+DISABLE_AA,
+MOVE_TO, 13, 3,
+LINE_TO, 14, 3,
+LINE_TO, 14, 4,
+LINE_TO, 13, 4,
+CLOSE,
+MOVE_TO, 13, 4,
+LINE_TO, 14, 4,
+LINE_TO, 14, 5,
+LINE_TO, 13, 5,
+CLOSE,
+MOVE_TO, 14, 5,
+LINE_TO, 15, 5,
+LINE_TO, 15, 6,
+LINE_TO, 14, 6,
+CLOSE,
+MOVE_TO, 14, 6,
+LINE_TO, 15, 6,
+LINE_TO, 15, 7,
+LINE_TO, 14, 7,
+CLOSE,
+MOVE_TO, 14, 7,
+LINE_TO, 15, 7,
+LINE_TO, 15, 8,
+LINE_TO, 14, 8,
+CLOSE,
+MOVE_TO, 14, 8,
+LINE_TO, 15, 8,
+LINE_TO, 15, 9,
+LINE_TO, 14, 9,
+CLOSE,
+MOVE_TO, 14, 9,
+LINE_TO, 15, 9,
+LINE_TO, 15, 10,
+LINE_TO, 14, 10,
+CLOSE,
+MOVE_TO, 13, 10,
+LINE_TO, 15, 10,
+LINE_TO, 15, 11,
+LINE_TO, 13, 11,
+CLOSE,
+MOVE_TO, 13, 11,
+LINE_TO, 14, 11,
+LINE_TO, 14, 12,
+LINE_TO, 13, 12,
+CLOSE,
+MOVE_TO, 3, 13,
+LINE_TO, 5, 13,
+LINE_TO, 5, 14,
+LINE_TO, 3, 14,
+CLOSE,
+MOVE_TO, 10, 13,
+LINE_TO, 12, 13,
+LINE_TO, 12, 14,
+LINE_TO, 10, 14,
+CLOSE,
+MOVE_TO, 5, 14,
+LINE_TO, 10, 14,
+LINE_TO, 10, 15,
+LINE_TO, 5, 15,
+CLOSE,
+NEW_PATH,
+FILL_RULE_NONZERO,
+DISABLE_AA,
+MOVE_TO, 5, 1,
+LINE_TO, 10, 1,
+LINE_TO, 10, 2,
+LINE_TO, 5, 2,
+CLOSE,
+MOVE_TO, 3, 2,
+LINE_TO, 5, 2,
+LINE_TO, 5, 3,
+LINE_TO, 3, 3,
+CLOSE,
+MOVE_TO, 10, 2,
+LINE_TO, 12, 2,
+LINE_TO, 12, 3,
+LINE_TO, 10, 3,
+CLOSE,
+MOVE_TO, 2, 3,
+LINE_TO, 3, 3,
+LINE_TO, 3, 4,
+LINE_TO, 2, 4,
+CLOSE,
+MOVE_TO, 12, 3,
+LINE_TO, 13, 3,
+LINE_TO, 13, 4,
+LINE_TO, 12, 4,
+CLOSE,
+MOVE_TO, 2, 4,
+LINE_TO, 3, 4,
+LINE_TO, 3, 5,
+LINE_TO, 2, 5,
+CLOSE,
+MOVE_TO, 7, 4,
+LINE_TO, 8, 4,
+LINE_TO, 8, 5,
+LINE_TO, 7, 5,
+CLOSE,
+MOVE_TO, 12, 4,
+LINE_TO, 13, 4,
+LINE_TO, 13, 5,
+LINE_TO, 12, 5,
+CLOSE,
+MOVE_TO, 1, 5,
+LINE_TO, 2, 5,
+LINE_TO, 2, 6,
+LINE_TO, 1, 6,
+CLOSE,
+MOVE_TO, 13, 5,
+LINE_TO, 14, 5,
+LINE_TO, 14, 6,
+LINE_TO, 13, 6,
+CLOSE,
+MOVE_TO, 1, 6,
+LINE_TO, 2, 6,
+LINE_TO, 2, 7,
+LINE_TO, 1, 7,
+CLOSE,
+MOVE_TO, 7, 6,
+LINE_TO, 8, 6,
+LINE_TO, 8, 7,
+LINE_TO, 7, 7,
+CLOSE,
+MOVE_TO, 13, 6,
+LINE_TO, 14, 6,
+LINE_TO, 14, 7,
+LINE_TO, 13, 7,
+CLOSE,
+MOVE_TO, 1, 7,
+LINE_TO, 2, 7,
+LINE_TO, 2, 8,
+LINE_TO, 1, 8,
+CLOSE,
+MOVE_TO, 7, 7,
+LINE_TO, 8, 7,
+LINE_TO, 8, 8,
+LINE_TO, 7, 8,
+CLOSE,
+MOVE_TO, 13, 7,
+LINE_TO, 14, 7,
+LINE_TO, 14, 8,
+LINE_TO, 13, 8,
+CLOSE,
+MOVE_TO, 1, 8,
+LINE_TO, 2, 8,
+LINE_TO, 2, 9,
+LINE_TO, 1, 9,
+CLOSE,
+MOVE_TO, 7, 8,
+LINE_TO, 8, 8,
+LINE_TO, 8, 9,
+LINE_TO, 7, 9,
+CLOSE,
+MOVE_TO, 13, 8,
+LINE_TO, 14, 8,
+LINE_TO, 14, 9,
+LINE_TO, 13, 9,
+CLOSE,
+MOVE_TO, 1, 9,
+LINE_TO, 2, 9,
+LINE_TO, 2, 10,
+LINE_TO, 1, 10,
+CLOSE,
+MOVE_TO, 7, 9,
+LINE_TO, 8, 9,
+LINE_TO, 8, 10,
+LINE_TO, 7, 10,
+CLOSE,
+MOVE_TO, 13, 9,
+LINE_TO, 14, 9,
+LINE_TO, 14, 10,
+LINE_TO, 13, 10,
+CLOSE,
+MOVE_TO, 2, 10,
+LINE_TO, 3, 10,
+LINE_TO, 3, 11,
+LINE_TO, 2, 11,
+CLOSE,
+MOVE_TO, 7, 10,
+LINE_TO, 8, 10,
+LINE_TO, 8, 11,
+LINE_TO, 7, 11,
+CLOSE,
+MOVE_TO, 12, 10,
+LINE_TO, 13, 10,
+LINE_TO, 13, 11,
+LINE_TO, 12, 11,
+CLOSE,
+MOVE_TO, 2, 11,
+LINE_TO, 3, 11,
+LINE_TO, 3, 12,
+LINE_TO, 2, 12,
+CLOSE,
+MOVE_TO, 12, 11,
+LINE_TO, 13, 11,
+LINE_TO, 13, 12,
+LINE_TO, 12, 12,
+CLOSE,
+MOVE_TO, 3, 12,
+LINE_TO, 5, 12,
+LINE_TO, 5, 13,
+LINE_TO, 3, 13,
+CLOSE,
+MOVE_TO, 10, 12,
+LINE_TO, 12, 12,
+LINE_TO, 12, 13,
+LINE_TO, 10, 13,
+CLOSE,
+MOVE_TO, 5, 13,
+LINE_TO, 10, 13,
+LINE_TO, 10, 14,
+LINE_TO, 5, 14,
+CLOSE,
--- /dev/null
+++ b/chrome/browser/ui/aerographene_theme_icons/pixelized_link.icon
@@ -0,0 +1,82 @@
+// Copyright 2026 The aerographene Authors
+// Use of this source code is governed by the GPL-3.0 license.
+// Gerado por theming/design-language/gen_theme_icons.py; não editar.
+
+CANVAS_DIMENSIONS, 16,
+FILL_RULE_NONZERO,
+DISABLE_AA,
+MOVE_TO, 2, 4,
+LINE_TO, 6, 4,
+LINE_TO, 6, 5,
+LINE_TO, 2, 5,
+CLOSE,
+MOVE_TO, 9, 4,
+LINE_TO, 13, 4,
+LINE_TO, 13, 5,
+LINE_TO, 9, 5,
+CLOSE,
+MOVE_TO, 1, 5,
+LINE_TO, 2, 5,
+LINE_TO, 2, 6,
+LINE_TO, 1, 6,
+CLOSE,
+MOVE_TO, 13, 5,
+LINE_TO, 14, 5,
+LINE_TO, 14, 6,
+LINE_TO, 13, 6,
+CLOSE,
+MOVE_TO, 1, 6,
+LINE_TO, 2, 6,
+LINE_TO, 2, 7,
+LINE_TO, 1, 7,
+CLOSE,
+MOVE_TO, 13, 6,
+LINE_TO, 14, 6,
+LINE_TO, 14, 7,
+LINE_TO, 13, 7,
+CLOSE,
+MOVE_TO, 1, 7,
+LINE_TO, 2, 7,
+LINE_TO, 2, 8,
+LINE_TO, 1, 8,
+CLOSE,
+MOVE_TO, 5, 7,
+LINE_TO, 10, 7,
+LINE_TO, 10, 8,
+LINE_TO, 5, 8,
+CLOSE,
+MOVE_TO, 13, 7,
+LINE_TO, 14, 7,
+LINE_TO, 14, 8,
+LINE_TO, 13, 8,
+CLOSE,
+MOVE_TO, 1, 8,
+LINE_TO, 2, 8,
+LINE_TO, 2, 9,
+LINE_TO, 1, 9,
+CLOSE,
+MOVE_TO, 13, 8,
+LINE_TO, 14, 8,
+LINE_TO, 14, 9,
+LINE_TO, 13, 9,
+CLOSE,
+MOVE_TO, 1, 9,
+LINE_TO, 2, 9,
+LINE_TO, 2, 10,
+LINE_TO, 1, 10,
+CLOSE,
+MOVE_TO, 13, 9,
+LINE_TO, 14, 9,
+LINE_TO, 14, 10,
+LINE_TO, 13, 10,
+CLOSE,
+MOVE_TO, 2, 10,
+LINE_TO, 6, 10,
+LINE_TO, 6, 11,
+LINE_TO, 2, 11,
+CLOSE,
+MOVE_TO, 9, 10,
+LINE_TO, 13, 10,
+LINE_TO, 13, 11,
+LINE_TO, 9, 11,
+CLOSE,
--- /dev/null
+++ b/chrome/browser/ui/aerographene_theme_icons/pixelized_max.icon
@@ -0,0 +1,210 @@
+// Copyright 2026 The aerographene Authors
+// Use of this source code is governed by the GPL-3.0 license.
+// Gerado por theming/design-language/gen_theme_icons.py; não editar.
+
+CANVAS_DIMENSIONS, 16,
+PATH_COLOR_ALPHA, 0x38,
+FILL_RULE_NONZERO,
+DISABLE_AA,
+MOVE_TO, 3, 3,
+LINE_TO, 12, 3,
+LINE_TO, 12, 4,
+LINE_TO, 3, 4,
+CLOSE,
+MOVE_TO, 3, 4,
+LINE_TO, 12, 4,
+LINE_TO, 12, 5,
+LINE_TO, 3, 5,
+CLOSE,
+MOVE_TO, 3, 5,
+LINE_TO, 12, 5,
+LINE_TO, 12, 6,
+LINE_TO, 3, 6,
+CLOSE,
+MOVE_TO, 3, 6,
+LINE_TO, 12, 6,
+LINE_TO, 12, 7,
+LINE_TO, 3, 7,
+CLOSE,
+MOVE_TO, 3, 7,
+LINE_TO, 12, 7,
+LINE_TO, 12, 8,
+LINE_TO, 3, 8,
+CLOSE,
+MOVE_TO, 3, 8,
+LINE_TO, 12, 8,
+LINE_TO, 12, 9,
+LINE_TO, 3, 9,
+CLOSE,
+MOVE_TO, 3, 9,
+LINE_TO, 12, 9,
+LINE_TO, 12, 10,
+LINE_TO, 3, 10,
+CLOSE,
+MOVE_TO, 3, 10,
+LINE_TO, 12, 10,
+LINE_TO, 12, 11,
+LINE_TO, 3, 11,
+CLOSE,
+MOVE_TO, 3, 11,
+LINE_TO, 12, 11,
+LINE_TO, 12, 12,
+LINE_TO, 3, 12,
+CLOSE,
+NEW_PATH,
+PATH_COLOR_ALPHA, 0x80,
+FILL_RULE_NONZERO,
+DISABLE_AA,
+MOVE_TO, 13, 3,
+LINE_TO, 14, 3,
+LINE_TO, 14, 4,
+LINE_TO, 13, 4,
+CLOSE,
+MOVE_TO, 13, 4,
+LINE_TO, 14, 4,
+LINE_TO, 14, 5,
+LINE_TO, 13, 5,
+CLOSE,
+MOVE_TO, 13, 5,
+LINE_TO, 14, 5,
+LINE_TO, 14, 6,
+LINE_TO, 13, 6,
+CLOSE,
+MOVE_TO, 13, 6,
+LINE_TO, 14, 6,
+LINE_TO, 14, 7,
+LINE_TO, 13, 7,
+CLOSE,
+MOVE_TO, 13, 7,
+LINE_TO, 14, 7,
+LINE_TO, 14, 8,
+LINE_TO, 13, 8,
+CLOSE,
+MOVE_TO, 13, 8,
+LINE_TO, 14, 8,
+LINE_TO, 14, 9,
+LINE_TO, 13, 9,
+CLOSE,
+MOVE_TO, 13, 9,
+LINE_TO, 14, 9,
+LINE_TO, 14, 10,
+LINE_TO, 13, 10,
+CLOSE,
+MOVE_TO, 13, 10,
+LINE_TO, 14, 10,
+LINE_TO, 14, 11,
+LINE_TO, 13, 11,
+CLOSE,
+MOVE_TO, 13, 11,
+LINE_TO, 14, 11,
+LINE_TO, 14, 12,
+LINE_TO, 13, 12,
+CLOSE,
+MOVE_TO, 3, 13,
+LINE_TO, 12, 13,
+LINE_TO, 12, 14,
+LINE_TO, 3, 14,
+CLOSE,
+NEW_PATH,
+FILL_RULE_NONZERO,
+DISABLE_AA,
+MOVE_TO, 2, 2,
+LINE_TO, 13, 2,
+LINE_TO, 13, 3,
+LINE_TO, 2, 3,
+CLOSE,
+MOVE_TO, 2, 3,
+LINE_TO, 3, 3,
+LINE_TO, 3, 4,
+LINE_TO, 2, 4,
+CLOSE,
+MOVE_TO, 12, 3,
+LINE_TO, 13, 3,
+LINE_TO, 13, 4,
+LINE_TO, 12, 4,
+CLOSE,
+MOVE_TO, 2, 4,
+LINE_TO, 3, 4,
+LINE_TO, 3, 5,
+LINE_TO, 2, 5,
+CLOSE,
+MOVE_TO, 12, 4,
+LINE_TO, 13, 4,
+LINE_TO, 13, 5,
+LINE_TO, 12, 5,
+CLOSE,
+MOVE_TO, 2, 5,
+LINE_TO, 3, 5,
+LINE_TO, 3, 6,
+LINE_TO, 2, 6,
+CLOSE,
+MOVE_TO, 12, 5,
+LINE_TO, 13, 5,
+LINE_TO, 13, 6,
+LINE_TO, 12, 6,
+CLOSE,
+MOVE_TO, 2, 6,
+LINE_TO, 3, 6,
+LINE_TO, 3, 7,
+LINE_TO, 2, 7,
+CLOSE,
+MOVE_TO, 12, 6,
+LINE_TO, 13, 6,
+LINE_TO, 13, 7,
+LINE_TO, 12, 7,
+CLOSE,
+MOVE_TO, 2, 7,
+LINE_TO, 3, 7,
+LINE_TO, 3, 8,
+LINE_TO, 2, 8,
+CLOSE,
+MOVE_TO, 12, 7,
+LINE_TO, 13, 7,
+LINE_TO, 13, 8,
+LINE_TO, 12, 8,
+CLOSE,
+MOVE_TO, 2, 8,
+LINE_TO, 3, 8,
+LINE_TO, 3, 9,
+LINE_TO, 2, 9,
+CLOSE,
+MOVE_TO, 12, 8,
+LINE_TO, 13, 8,
+LINE_TO, 13, 9,
+LINE_TO, 12, 9,
+CLOSE,
+MOVE_TO, 2, 9,
+LINE_TO, 3, 9,
+LINE_TO, 3, 10,
+LINE_TO, 2, 10,
+CLOSE,
+MOVE_TO, 12, 9,
+LINE_TO, 13, 9,
+LINE_TO, 13, 10,
+LINE_TO, 12, 10,
+CLOSE,
+MOVE_TO, 2, 10,
+LINE_TO, 3, 10,
+LINE_TO, 3, 11,
+LINE_TO, 2, 11,
+CLOSE,
+MOVE_TO, 12, 10,
+LINE_TO, 13, 10,
+LINE_TO, 13, 11,
+LINE_TO, 12, 11,
+CLOSE,
+MOVE_TO, 2, 11,
+LINE_TO, 3, 11,
+LINE_TO, 3, 12,
+LINE_TO, 2, 12,
+CLOSE,
+MOVE_TO, 12, 11,
+LINE_TO, 13, 11,
+LINE_TO, 13, 12,
+LINE_TO, 12, 12,
+CLOSE,
+MOVE_TO, 2, 12,
+LINE_TO, 13, 12,
+LINE_TO, 13, 13,
+LINE_TO, 2, 13,
+CLOSE,
--- /dev/null
+++ b/chrome/browser/ui/aerographene_theme_icons/pixelized_min.icon
@@ -0,0 +1,12 @@
+// Copyright 2026 The aerographene Authors
+// Use of this source code is governed by the GPL-3.0 license.
+// Gerado por theming/design-language/gen_theme_icons.py; não editar.
+
+CANVAS_DIMENSIONS, 16,
+FILL_RULE_NONZERO,
+DISABLE_AA,
+MOVE_TO, 2, 7,
+LINE_TO, 13, 7,
+LINE_TO, 13, 8,
+LINE_TO, 2, 8,
+CLOSE,
--- /dev/null
+++ b/chrome/browser/ui/aerographene_theme_icons/pixelized_minus.icon
@@ -0,0 +1,12 @@
+// Copyright 2026 The aerographene Authors
+// Use of this source code is governed by the GPL-3.0 license.
+// Gerado por theming/design-language/gen_theme_icons.py; não editar.
+
+CANVAS_DIMENSIONS, 16,
+FILL_RULE_NONZERO,
+DISABLE_AA,
+MOVE_TO, 2, 7,
+LINE_TO, 13, 7,
+LINE_TO, 13, 8,
+LINE_TO, 2, 8,
+CLOSE,
--- /dev/null
+++ b/chrome/browser/ui/aerographene_theme_icons/pixelized_mon.icon
@@ -0,0 +1,210 @@
+// Copyright 2026 The aerographene Authors
+// Use of this source code is governed by the GPL-3.0 license.
+// Gerado por theming/design-language/gen_theme_icons.py; não editar.
+
+CANVAS_DIMENSIONS, 16,
+PATH_COLOR_ALPHA, 0x38,
+FILL_RULE_NONZERO,
+DISABLE_AA,
+MOVE_TO, 2, 2,
+LINE_TO, 13, 2,
+LINE_TO, 13, 3,
+LINE_TO, 2, 3,
+CLOSE,
+MOVE_TO, 2, 3,
+LINE_TO, 13, 3,
+LINE_TO, 13, 4,
+LINE_TO, 2, 4,
+CLOSE,
+MOVE_TO, 2, 4,
+LINE_TO, 13, 4,
+LINE_TO, 13, 5,
+LINE_TO, 2, 5,
+CLOSE,
+MOVE_TO, 2, 5,
+LINE_TO, 13, 5,
+LINE_TO, 13, 6,
+LINE_TO, 2, 6,
+CLOSE,
+MOVE_TO, 2, 6,
+LINE_TO, 13, 6,
+LINE_TO, 13, 7,
+LINE_TO, 2, 7,
+CLOSE,
+MOVE_TO, 2, 7,
+LINE_TO, 13, 7,
+LINE_TO, 13, 8,
+LINE_TO, 2, 8,
+CLOSE,
+MOVE_TO, 2, 8,
+LINE_TO, 13, 8,
+LINE_TO, 13, 9,
+LINE_TO, 2, 9,
+CLOSE,
+MOVE_TO, 2, 9,
+LINE_TO, 13, 9,
+LINE_TO, 13, 10,
+LINE_TO, 2, 10,
+CLOSE,
+NEW_PATH,
+PATH_COLOR_ALPHA, 0x80,
+FILL_RULE_NONZERO,
+DISABLE_AA,
+MOVE_TO, 14, 2,
+LINE_TO, 15, 2,
+LINE_TO, 15, 3,
+LINE_TO, 14, 3,
+CLOSE,
+MOVE_TO, 14, 3,
+LINE_TO, 15, 3,
+LINE_TO, 15, 4,
+LINE_TO, 14, 4,
+CLOSE,
+MOVE_TO, 14, 4,
+LINE_TO, 15, 4,
+LINE_TO, 15, 5,
+LINE_TO, 14, 5,
+CLOSE,
+MOVE_TO, 14, 5,
+LINE_TO, 15, 5,
+LINE_TO, 15, 6,
+LINE_TO, 14, 6,
+CLOSE,
+MOVE_TO, 14, 6,
+LINE_TO, 15, 6,
+LINE_TO, 15, 7,
+LINE_TO, 14, 7,
+CLOSE,
+MOVE_TO, 14, 7,
+LINE_TO, 15, 7,
+LINE_TO, 15, 8,
+LINE_TO, 14, 8,
+CLOSE,
+MOVE_TO, 14, 8,
+LINE_TO, 15, 8,
+LINE_TO, 15, 9,
+LINE_TO, 14, 9,
+CLOSE,
+MOVE_TO, 14, 9,
+LINE_TO, 15, 9,
+LINE_TO, 15, 10,
+LINE_TO, 14, 10,
+CLOSE,
+MOVE_TO, 2, 11,
+LINE_TO, 7, 11,
+LINE_TO, 7, 12,
+LINE_TO, 2, 12,
+CLOSE,
+MOVE_TO, 8, 11,
+LINE_TO, 13, 11,
+LINE_TO, 13, 12,
+LINE_TO, 8, 12,
+CLOSE,
+NEW_PATH,
+FILL_RULE_NONZERO,
+DISABLE_AA,
+MOVE_TO, 1, 1,
+LINE_TO, 14, 1,
+LINE_TO, 14, 2,
+LINE_TO, 1, 2,
+CLOSE,
+MOVE_TO, 1, 2,
+LINE_TO, 2, 2,
+LINE_TO, 2, 3,
+LINE_TO, 1, 3,
+CLOSE,
+MOVE_TO, 13, 2,
+LINE_TO, 14, 2,
+LINE_TO, 14, 3,
+LINE_TO, 13, 3,
+CLOSE,
+MOVE_TO, 1, 3,
+LINE_TO, 2, 3,
+LINE_TO, 2, 4,
+LINE_TO, 1, 4,
+CLOSE,
+MOVE_TO, 13, 3,
+LINE_TO, 14, 3,
+LINE_TO, 14, 4,
+LINE_TO, 13, 4,
+CLOSE,
+MOVE_TO, 1, 4,
+LINE_TO, 2, 4,
+LINE_TO, 2, 5,
+LINE_TO, 1, 5,
+CLOSE,
+MOVE_TO, 13, 4,
+LINE_TO, 14, 4,
+LINE_TO, 14, 5,
+LINE_TO, 13, 5,
+CLOSE,
+MOVE_TO, 1, 5,
+LINE_TO, 2, 5,
+LINE_TO, 2, 6,
+LINE_TO, 1, 6,
+CLOSE,
+MOVE_TO, 13, 5,
+LINE_TO, 14, 5,
+LINE_TO, 14, 6,
+LINE_TO, 13, 6,
+CLOSE,
+MOVE_TO, 1, 6,
+LINE_TO, 2, 6,
+LINE_TO, 2, 7,
+LINE_TO, 1, 7,
+CLOSE,
+MOVE_TO, 13, 6,
+LINE_TO, 14, 6,
+LINE_TO, 14, 7,
+LINE_TO, 13, 7,
+CLOSE,
+MOVE_TO, 1, 7,
+LINE_TO, 2, 7,
+LINE_TO, 2, 8,
+LINE_TO, 1, 8,
+CLOSE,
+MOVE_TO, 13, 7,
+LINE_TO, 14, 7,
+LINE_TO, 14, 8,
+LINE_TO, 13, 8,
+CLOSE,
+MOVE_TO, 1, 8,
+LINE_TO, 2, 8,
+LINE_TO, 2, 9,
+LINE_TO, 1, 9,
+CLOSE,
+MOVE_TO, 13, 8,
+LINE_TO, 14, 8,
+LINE_TO, 14, 9,
+LINE_TO, 13, 9,
+CLOSE,
+MOVE_TO, 1, 9,
+LINE_TO, 2, 9,
+LINE_TO, 2, 10,
+LINE_TO, 1, 10,
+CLOSE,
+MOVE_TO, 13, 9,
+LINE_TO, 14, 9,
+LINE_TO, 14, 10,
+LINE_TO, 13, 10,
+CLOSE,
+MOVE_TO, 1, 10,
+LINE_TO, 14, 10,
+LINE_TO, 14, 11,
+LINE_TO, 1, 11,
+CLOSE,
+MOVE_TO, 7, 11,
+LINE_TO, 8, 11,
+LINE_TO, 8, 12,
+LINE_TO, 7, 12,
+CLOSE,
+MOVE_TO, 6, 12,
+LINE_TO, 9, 12,
+LINE_TO, 9, 13,
+LINE_TO, 6, 13,
+CLOSE,
+MOVE_TO, 4, 13,
+LINE_TO, 11, 13,
+LINE_TO, 11, 14,
+LINE_TO, 4, 14,
+CLOSE,
--- /dev/null
+++ b/chrome/browser/ui/aerographene_theme_icons/pixelized_more.icon
@@ -0,0 +1,37 @@
+// Copyright 2026 The aerographene Authors
+// Use of this source code is governed by the GPL-3.0 license.
+// Gerado por theming/design-language/gen_theme_icons.py; não editar.
+
+CANVAS_DIMENSIONS, 16,
+FILL_RULE_NONZERO,
+DISABLE_AA,
+MOVE_TO, 3, 7,
+LINE_TO, 5, 7,
+LINE_TO, 5, 8,
+LINE_TO, 3, 8,
+CLOSE,
+MOVE_TO, 7, 7,
+LINE_TO, 9, 7,
+LINE_TO, 9, 8,
+LINE_TO, 7, 8,
+CLOSE,
+MOVE_TO, 11, 7,
+LINE_TO, 13, 7,
+LINE_TO, 13, 8,
+LINE_TO, 11, 8,
+CLOSE,
+MOVE_TO, 3, 8,
+LINE_TO, 5, 8,
+LINE_TO, 5, 9,
+LINE_TO, 3, 9,
+CLOSE,
+MOVE_TO, 7, 8,
+LINE_TO, 9, 8,
+LINE_TO, 9, 9,
+LINE_TO, 7, 9,
+CLOSE,
+MOVE_TO, 11, 8,
+LINE_TO, 13, 8,
+LINE_TO, 13, 9,
+LINE_TO, 11, 9,
+CLOSE,
--- /dev/null
+++ b/chrome/browser/ui/aerographene_theme_icons/pixelized_plus.icon
@@ -0,0 +1,62 @@
+// Copyright 2026 The aerographene Authors
+// Use of this source code is governed by the GPL-3.0 license.
+// Gerado por theming/design-language/gen_theme_icons.py; não editar.
+
+CANVAS_DIMENSIONS, 16,
+FILL_RULE_NONZERO,
+DISABLE_AA,
+MOVE_TO, 7, 2,
+LINE_TO, 8, 2,
+LINE_TO, 8, 3,
+LINE_TO, 7, 3,
+CLOSE,
+MOVE_TO, 7, 3,
+LINE_TO, 8, 3,
+LINE_TO, 8, 4,
+LINE_TO, 7, 4,
+CLOSE,
+MOVE_TO, 7, 4,
+LINE_TO, 8, 4,
+LINE_TO, 8, 5,
+LINE_TO, 7, 5,
+CLOSE,
+MOVE_TO, 7, 5,
+LINE_TO, 8, 5,
+LINE_TO, 8, 6,
+LINE_TO, 7, 6,
+CLOSE,
+MOVE_TO, 7, 6,
+LINE_TO, 8, 6,
+LINE_TO, 8, 7,
+LINE_TO, 7, 7,
+CLOSE,
+MOVE_TO, 2, 7,
+LINE_TO, 13, 7,
+LINE_TO, 13, 8,
+LINE_TO, 2, 8,
+CLOSE,
+MOVE_TO, 7, 8,
+LINE_TO, 8, 8,
+LINE_TO, 8, 9,
+LINE_TO, 7, 9,
+CLOSE,
+MOVE_TO, 7, 9,
+LINE_TO, 8, 9,
+LINE_TO, 8, 10,
+LINE_TO, 7, 10,
+CLOSE,
+MOVE_TO, 7, 10,
+LINE_TO, 8, 10,
+LINE_TO, 8, 11,
+LINE_TO, 7, 11,
+CLOSE,
+MOVE_TO, 7, 11,
+LINE_TO, 8, 11,
+LINE_TO, 8, 12,
+LINE_TO, 7, 12,
+CLOSE,
+MOVE_TO, 7, 12,
+LINE_TO, 8, 12,
+LINE_TO, 8, 13,
+LINE_TO, 7, 13,
+CLOSE,
--- /dev/null
+++ b/chrome/browser/ui/aerographene_theme_icons/pixelized_rel.icon
@@ -0,0 +1,140 @@
+// Copyright 2026 The aerographene Authors
+// Use of this source code is governed by the GPL-3.0 license.
+// Gerado por theming/design-language/gen_theme_icons.py; não editar.
+
+CANVAS_DIMENSIONS, 16,
+PATH_COLOR_ALPHA, 0x38,
+FILL_RULE_NONZERO,
+DISABLE_AA,
+MOVE_TO, 9, 4,
+LINE_TO, 11, 4,
+LINE_TO, 11, 5,
+LINE_TO, 9, 5,
+CLOSE,
+NEW_PATH,
+PATH_COLOR_ALPHA, 0x80,
+FILL_RULE_NONZERO,
+DISABLE_AA,
+MOVE_TO, 12, 4,
+LINE_TO, 13, 4,
+LINE_TO, 13, 5,
+LINE_TO, 12, 5,
+CLOSE,
+MOVE_TO, 9, 6,
+LINE_TO, 11, 6,
+LINE_TO, 11, 7,
+LINE_TO, 9, 7,
+CLOSE,
+NEW_PATH,
+FILL_RULE_NONZERO,
+DISABLE_AA,
+MOVE_TO, 8, 2,
+LINE_TO, 9, 2,
+LINE_TO, 9, 3,
+LINE_TO, 8, 3,
+CLOSE,
+MOVE_TO, 5, 3,
+LINE_TO, 11, 3,
+LINE_TO, 11, 4,
+LINE_TO, 5, 4,
+CLOSE,
+MOVE_TO, 4, 4,
+LINE_TO, 5, 4,
+LINE_TO, 5, 5,
+LINE_TO, 4, 5,
+CLOSE,
+MOVE_TO, 8, 4,
+LINE_TO, 9, 4,
+LINE_TO, 9, 5,
+LINE_TO, 8, 5,
+CLOSE,
+MOVE_TO, 11, 4,
+LINE_TO, 12, 4,
+LINE_TO, 12, 5,
+LINE_TO, 11, 5,
+CLOSE,
+MOVE_TO, 3, 5,
+LINE_TO, 4, 5,
+LINE_TO, 4, 6,
+LINE_TO, 3, 6,
+CLOSE,
+MOVE_TO, 8, 5,
+LINE_TO, 11, 5,
+LINE_TO, 11, 6,
+LINE_TO, 8, 6,
+CLOSE,
+MOVE_TO, 2, 6,
+LINE_TO, 3, 6,
+LINE_TO, 3, 7,
+LINE_TO, 2, 7,
+CLOSE,
+MOVE_TO, 8, 6,
+LINE_TO, 9, 6,
+LINE_TO, 9, 7,
+LINE_TO, 8, 7,
+CLOSE,
+MOVE_TO, 2, 7,
+LINE_TO, 3, 7,
+LINE_TO, 3, 8,
+LINE_TO, 2, 8,
+CLOSE,
+MOVE_TO, 12, 7,
+LINE_TO, 13, 7,
+LINE_TO, 13, 8,
+LINE_TO, 12, 8,
+CLOSE,
+MOVE_TO, 2, 8,
+LINE_TO, 3, 8,
+LINE_TO, 3, 9,
+LINE_TO, 2, 9,
+CLOSE,
+MOVE_TO, 12, 8,
+LINE_TO, 13, 8,
+LINE_TO, 13, 9,
+LINE_TO, 12, 9,
+CLOSE,
+MOVE_TO, 2, 9,
+LINE_TO, 3, 9,
+LINE_TO, 3, 10,
+LINE_TO, 2, 10,
+CLOSE,
+MOVE_TO, 12, 9,
+LINE_TO, 13, 9,
+LINE_TO, 13, 10,
+LINE_TO, 12, 10,
+CLOSE,
+MOVE_TO, 2, 10,
+LINE_TO, 3, 10,
+LINE_TO, 3, 11,
+LINE_TO, 2, 11,
+CLOSE,
+MOVE_TO, 12, 10,
+LINE_TO, 13, 10,
+LINE_TO, 13, 11,
+LINE_TO, 12, 11,
+CLOSE,
+MOVE_TO, 3, 11,
+LINE_TO, 4, 11,
+LINE_TO, 4, 12,
+LINE_TO, 3, 12,
+CLOSE,
+MOVE_TO, 11, 11,
+LINE_TO, 12, 11,
+LINE_TO, 12, 12,
+LINE_TO, 11, 12,
+CLOSE,
+MOVE_TO, 4, 12,
+LINE_TO, 5, 12,
+LINE_TO, 5, 13,
+LINE_TO, 4, 13,
+CLOSE,
+MOVE_TO, 10, 12,
+LINE_TO, 11, 12,
+LINE_TO, 11, 13,
+LINE_TO, 10, 13,
+CLOSE,
+MOVE_TO, 5, 13,
+LINE_TO, 10, 13,
+LINE_TO, 10, 14,
+LINE_TO, 5, 14,
+CLOSE,
--- /dev/null
+++ b/chrome/browser/ui/aerographene_theme_icons/pixelized_search.icon
@@ -0,0 +1,190 @@
+// Copyright 2026 The aerographene Authors
+// Use of this source code is governed by the GPL-3.0 license.
+// Gerado por theming/design-language/gen_theme_icons.py; não editar.
+
+CANVAS_DIMENSIONS, 16,
+PATH_COLOR_ALPHA, 0x38,
+FILL_RULE_NONZERO,
+DISABLE_AA,
+MOVE_TO, 5, 3,
+LINE_TO, 8, 3,
+LINE_TO, 8, 4,
+LINE_TO, 5, 4,
+CLOSE,
+MOVE_TO, 4, 4,
+LINE_TO, 9, 4,
+LINE_TO, 9, 5,
+LINE_TO, 4, 5,
+CLOSE,
+MOVE_TO, 3, 5,
+LINE_TO, 10, 5,
+LINE_TO, 10, 6,
+LINE_TO, 3, 6,
+CLOSE,
+MOVE_TO, 3, 6,
+LINE_TO, 10, 6,
+LINE_TO, 10, 7,
+LINE_TO, 3, 7,
+CLOSE,
+MOVE_TO, 3, 7,
+LINE_TO, 10, 7,
+LINE_TO, 10, 8,
+LINE_TO, 3, 8,
+CLOSE,
+MOVE_TO, 4, 8,
+LINE_TO, 9, 8,
+LINE_TO, 9, 9,
+LINE_TO, 4, 9,
+CLOSE,
+MOVE_TO, 5, 9,
+LINE_TO, 8, 9,
+LINE_TO, 8, 10,
+LINE_TO, 5, 10,
+CLOSE,
+NEW_PATH,
+PATH_COLOR_ALPHA, 0x80,
+FILL_RULE_NONZERO,
+DISABLE_AA,
+MOVE_TO, 10, 4,
+LINE_TO, 11, 4,
+LINE_TO, 11, 5,
+LINE_TO, 10, 5,
+CLOSE,
+MOVE_TO, 11, 5,
+LINE_TO, 12, 5,
+LINE_TO, 12, 6,
+LINE_TO, 11, 6,
+CLOSE,
+MOVE_TO, 11, 6,
+LINE_TO, 12, 6,
+LINE_TO, 12, 7,
+LINE_TO, 11, 7,
+CLOSE,
+MOVE_TO, 11, 7,
+LINE_TO, 12, 7,
+LINE_TO, 12, 8,
+LINE_TO, 11, 8,
+CLOSE,
+MOVE_TO, 10, 8,
+LINE_TO, 12, 8,
+LINE_TO, 12, 9,
+LINE_TO, 10, 9,
+CLOSE,
+MOVE_TO, 4, 10,
+LINE_TO, 5, 10,
+LINE_TO, 5, 11,
+LINE_TO, 4, 11,
+CLOSE,
+MOVE_TO, 8, 10,
+LINE_TO, 9, 10,
+LINE_TO, 9, 11,
+LINE_TO, 8, 11,
+CLOSE,
+MOVE_TO, 5, 11,
+LINE_TO, 8, 11,
+LINE_TO, 8, 12,
+LINE_TO, 5, 12,
+CLOSE,
+NEW_PATH,
+FILL_RULE_NONZERO,
+DISABLE_AA,
+MOVE_TO, 5, 2,
+LINE_TO, 8, 2,
+LINE_TO, 8, 3,
+LINE_TO, 5, 3,
+CLOSE,
+MOVE_TO, 3, 3,
+LINE_TO, 5, 3,
+LINE_TO, 5, 4,
+LINE_TO, 3, 4,
+CLOSE,
+MOVE_TO, 8, 3,
+LINE_TO, 10, 3,
+LINE_TO, 10, 4,
+LINE_TO, 8, 4,
+CLOSE,
+MOVE_TO, 3, 4,
+LINE_TO, 4, 4,
+LINE_TO, 4, 5,
+LINE_TO, 3, 5,
+CLOSE,
+MOVE_TO, 9, 4,
+LINE_TO, 10, 4,
+LINE_TO, 10, 5,
+LINE_TO, 9, 5,
+CLOSE,
+MOVE_TO, 2, 5,
+LINE_TO, 3, 5,
+LINE_TO, 3, 6,
+LINE_TO, 2, 6,
+CLOSE,
+MOVE_TO, 10, 5,
+LINE_TO, 11, 5,
+LINE_TO, 11, 6,
+LINE_TO, 10, 6,
+CLOSE,
+MOVE_TO, 2, 6,
+LINE_TO, 3, 6,
+LINE_TO, 3, 7,
+LINE_TO, 2, 7,
+CLOSE,
+MOVE_TO, 10, 6,
+LINE_TO, 11, 6,
+LINE_TO, 11, 7,
+LINE_TO, 10, 7,
+CLOSE,
+MOVE_TO, 2, 7,
+LINE_TO, 3, 7,
+LINE_TO, 3, 8,
+LINE_TO, 2, 8,
+CLOSE,
+MOVE_TO, 10, 7,
+LINE_TO, 11, 7,
+LINE_TO, 11, 8,
+LINE_TO, 10, 8,
+CLOSE,
+MOVE_TO, 3, 8,
+LINE_TO, 4, 8,
+LINE_TO, 4, 9,
+LINE_TO, 3, 9,
+CLOSE,
+MOVE_TO, 9, 8,
+LINE_TO, 10, 8,
+LINE_TO, 10, 9,
+LINE_TO, 9, 9,
+CLOSE,
+MOVE_TO, 3, 9,
+LINE_TO, 5, 9,
+LINE_TO, 5, 10,
+LINE_TO, 3, 10,
+CLOSE,
+MOVE_TO, 8, 9,
+LINE_TO, 10, 9,
+LINE_TO, 10, 10,
+LINE_TO, 8, 10,
+CLOSE,
+MOVE_TO, 5, 10,
+LINE_TO, 8, 10,
+LINE_TO, 8, 11,
+LINE_TO, 5, 11,
+CLOSE,
+MOVE_TO, 10, 10,
+LINE_TO, 11, 10,
+LINE_TO, 11, 11,
+LINE_TO, 10, 11,
+CLOSE,
+MOVE_TO, 11, 11,
+LINE_TO, 12, 11,
+LINE_TO, 12, 12,
+LINE_TO, 11, 12,
+CLOSE,
+MOVE_TO, 12, 12,
+LINE_TO, 13, 12,
+LINE_TO, 13, 13,
+LINE_TO, 12, 13,
+CLOSE,
+MOVE_TO, 13, 13,
+LINE_TO, 14, 13,
+LINE_TO, 14, 14,
+LINE_TO, 13, 14,
+CLOSE,
--- /dev/null
+++ b/chrome/browser/ui/aerographene_theme_icons/pixelized_side.icon
@@ -0,0 +1,305 @@
+// Copyright 2026 The aerographene Authors
+// Use of this source code is governed by the GPL-3.0 license.
+// Gerado por theming/design-language/gen_theme_icons.py; não editar.
+
+CANVAS_DIMENSIONS, 16,
+PATH_COLOR_ALPHA, 0x38,
+FILL_RULE_NONZERO,
+DISABLE_AA,
+MOVE_TO, 2, 2,
+LINE_TO, 5, 2,
+LINE_TO, 5, 3,
+LINE_TO, 2, 3,
+CLOSE,
+MOVE_TO, 2, 3,
+LINE_TO, 5, 3,
+LINE_TO, 5, 4,
+LINE_TO, 2, 4,
+CLOSE,
+MOVE_TO, 2, 4,
+LINE_TO, 5, 4,
+LINE_TO, 5, 5,
+LINE_TO, 2, 5,
+CLOSE,
+MOVE_TO, 2, 5,
+LINE_TO, 5, 5,
+LINE_TO, 5, 6,
+LINE_TO, 2, 6,
+CLOSE,
+MOVE_TO, 2, 6,
+LINE_TO, 5, 6,
+LINE_TO, 5, 7,
+LINE_TO, 2, 7,
+CLOSE,
+MOVE_TO, 2, 7,
+LINE_TO, 5, 7,
+LINE_TO, 5, 8,
+LINE_TO, 2, 8,
+CLOSE,
+MOVE_TO, 2, 8,
+LINE_TO, 5, 8,
+LINE_TO, 5, 9,
+LINE_TO, 2, 9,
+CLOSE,
+MOVE_TO, 2, 9,
+LINE_TO, 5, 9,
+LINE_TO, 5, 10,
+LINE_TO, 2, 10,
+CLOSE,
+MOVE_TO, 2, 10,
+LINE_TO, 5, 10,
+LINE_TO, 5, 11,
+LINE_TO, 2, 11,
+CLOSE,
+MOVE_TO, 2, 11,
+LINE_TO, 5, 11,
+LINE_TO, 5, 12,
+LINE_TO, 2, 12,
+CLOSE,
+MOVE_TO, 2, 12,
+LINE_TO, 5, 12,
+LINE_TO, 5, 13,
+LINE_TO, 2, 13,
+CLOSE,
+NEW_PATH,
+PATH_COLOR_ALPHA, 0x80,
+FILL_RULE_NONZERO,
+DISABLE_AA,
+MOVE_TO, 6, 2,
+LINE_TO, 7, 2,
+LINE_TO, 7, 3,
+LINE_TO, 6, 3,
+CLOSE,
+MOVE_TO, 6, 3,
+LINE_TO, 7, 3,
+LINE_TO, 7, 4,
+LINE_TO, 6, 4,
+CLOSE,
+MOVE_TO, 6, 4,
+LINE_TO, 7, 4,
+LINE_TO, 7, 5,
+LINE_TO, 6, 5,
+CLOSE,
+MOVE_TO, 6, 5,
+LINE_TO, 7, 5,
+LINE_TO, 7, 6,
+LINE_TO, 6, 6,
+CLOSE,
+MOVE_TO, 6, 6,
+LINE_TO, 7, 6,
+LINE_TO, 7, 7,
+LINE_TO, 6, 7,
+CLOSE,
+MOVE_TO, 6, 7,
+LINE_TO, 7, 7,
+LINE_TO, 7, 8,
+LINE_TO, 6, 8,
+CLOSE,
+MOVE_TO, 6, 8,
+LINE_TO, 7, 8,
+LINE_TO, 7, 9,
+LINE_TO, 6, 9,
+CLOSE,
+MOVE_TO, 6, 9,
+LINE_TO, 7, 9,
+LINE_TO, 7, 10,
+LINE_TO, 6, 10,
+CLOSE,
+MOVE_TO, 6, 10,
+LINE_TO, 7, 10,
+LINE_TO, 7, 11,
+LINE_TO, 6, 11,
+CLOSE,
+MOVE_TO, 6, 11,
+LINE_TO, 7, 11,
+LINE_TO, 7, 12,
+LINE_TO, 6, 12,
+CLOSE,
+MOVE_TO, 6, 12,
+LINE_TO, 7, 12,
+LINE_TO, 7, 13,
+LINE_TO, 6, 13,
+CLOSE,
+MOVE_TO, 2, 14,
+LINE_TO, 5, 14,
+LINE_TO, 5, 15,
+LINE_TO, 2, 15,
+CLOSE,
+NEW_PATH,
+FILL_RULE_NONZERO,
+DISABLE_AA,
+MOVE_TO, 1, 1,
+LINE_TO, 14, 1,
+LINE_TO, 14, 2,
+LINE_TO, 1, 2,
+CLOSE,
+MOVE_TO, 1, 2,
+LINE_TO, 2, 2,
+LINE_TO, 2, 3,
+LINE_TO, 1, 3,
+CLOSE,
+MOVE_TO, 5, 2,
+LINE_TO, 6, 2,
+LINE_TO, 6, 3,
+LINE_TO, 5, 3,
+CLOSE,
+MOVE_TO, 13, 2,
+LINE_TO, 14, 2,
+LINE_TO, 14, 3,
+LINE_TO, 13, 3,
+CLOSE,
+MOVE_TO, 1, 3,
+LINE_TO, 2, 3,
+LINE_TO, 2, 4,
+LINE_TO, 1, 4,
+CLOSE,
+MOVE_TO, 5, 3,
+LINE_TO, 6, 3,
+LINE_TO, 6, 4,
+LINE_TO, 5, 4,
+CLOSE,
+MOVE_TO, 13, 3,
+LINE_TO, 14, 3,
+LINE_TO, 14, 4,
+LINE_TO, 13, 4,
+CLOSE,
+MOVE_TO, 1, 4,
+LINE_TO, 2, 4,
+LINE_TO, 2, 5,
+LINE_TO, 1, 5,
+CLOSE,
+MOVE_TO, 5, 4,
+LINE_TO, 6, 4,
+LINE_TO, 6, 5,
+LINE_TO, 5, 5,
+CLOSE,
+MOVE_TO, 13, 4,
+LINE_TO, 14, 4,
+LINE_TO, 14, 5,
+LINE_TO, 13, 5,
+CLOSE,
+MOVE_TO, 1, 5,
+LINE_TO, 2, 5,
+LINE_TO, 2, 6,
+LINE_TO, 1, 6,
+CLOSE,
+MOVE_TO, 5, 5,
+LINE_TO, 6, 5,
+LINE_TO, 6, 6,
+LINE_TO, 5, 6,
+CLOSE,
+MOVE_TO, 13, 5,
+LINE_TO, 14, 5,
+LINE_TO, 14, 6,
+LINE_TO, 13, 6,
+CLOSE,
+MOVE_TO, 1, 6,
+LINE_TO, 2, 6,
+LINE_TO, 2, 7,
+LINE_TO, 1, 7,
+CLOSE,
+MOVE_TO, 5, 6,
+LINE_TO, 6, 6,
+LINE_TO, 6, 7,
+LINE_TO, 5, 7,
+CLOSE,
+MOVE_TO, 13, 6,
+LINE_TO, 14, 6,
+LINE_TO, 14, 7,
+LINE_TO, 13, 7,
+CLOSE,
+MOVE_TO, 1, 7,
+LINE_TO, 2, 7,
+LINE_TO, 2, 8,
+LINE_TO, 1, 8,
+CLOSE,
+MOVE_TO, 5, 7,
+LINE_TO, 6, 7,
+LINE_TO, 6, 8,
+LINE_TO, 5, 8,
+CLOSE,
+MOVE_TO, 13, 7,
+LINE_TO, 14, 7,
+LINE_TO, 14, 8,
+LINE_TO, 13, 8,
+CLOSE,
+MOVE_TO, 1, 8,
+LINE_TO, 2, 8,
+LINE_TO, 2, 9,
+LINE_TO, 1, 9,
+CLOSE,
+MOVE_TO, 5, 8,
+LINE_TO, 6, 8,
+LINE_TO, 6, 9,
+LINE_TO, 5, 9,
+CLOSE,
+MOVE_TO, 13, 8,
+LINE_TO, 14, 8,
+LINE_TO, 14, 9,
+LINE_TO, 13, 9,
+CLOSE,
+MOVE_TO, 1, 9,
+LINE_TO, 2, 9,
+LINE_TO, 2, 10,
+LINE_TO, 1, 10,
+CLOSE,
+MOVE_TO, 5, 9,
+LINE_TO, 6, 9,
+LINE_TO, 6, 10,
+LINE_TO, 5, 10,
+CLOSE,
+MOVE_TO, 13, 9,
+LINE_TO, 14, 9,
+LINE_TO, 14, 10,
+LINE_TO, 13, 10,
+CLOSE,
+MOVE_TO, 1, 10,
+LINE_TO, 2, 10,
+LINE_TO, 2, 11,
+LINE_TO, 1, 11,
+CLOSE,
+MOVE_TO, 5, 10,
+LINE_TO, 6, 10,
+LINE_TO, 6, 11,
+LINE_TO, 5, 11,
+CLOSE,
+MOVE_TO, 13, 10,
+LINE_TO, 14, 10,
+LINE_TO, 14, 11,
+LINE_TO, 13, 11,
+CLOSE,
+MOVE_TO, 1, 11,
+LINE_TO, 2, 11,
+LINE_TO, 2, 12,
+LINE_TO, 1, 12,
+CLOSE,
+MOVE_TO, 5, 11,
+LINE_TO, 6, 11,
+LINE_TO, 6, 12,
+LINE_TO, 5, 12,
+CLOSE,
+MOVE_TO, 13, 11,
+LINE_TO, 14, 11,
+LINE_TO, 14, 12,
+LINE_TO, 13, 12,
+CLOSE,
+MOVE_TO, 1, 12,
+LINE_TO, 2, 12,
+LINE_TO, 2, 13,
+LINE_TO, 1, 13,
+CLOSE,
+MOVE_TO, 5, 12,
+LINE_TO, 6, 12,
+LINE_TO, 6, 13,
+LINE_TO, 5, 13,
+CLOSE,
+MOVE_TO, 13, 12,
+LINE_TO, 14, 12,
+LINE_TO, 14, 13,
+LINE_TO, 13, 13,
+CLOSE,
+MOVE_TO, 1, 13,
+LINE_TO, 14, 13,
+LINE_TO, 14, 14,
+LINE_TO, 1, 14,
+CLOSE,
--- /dev/null
+++ b/chrome/browser/ui/aerographene_theme_icons/pixelized_spk.icon
@@ -0,0 +1,305 @@
+// Copyright 2026 The aerographene Authors
+// Use of this source code is governed by the GPL-3.0 license.
+// Gerado por theming/design-language/gen_theme_icons.py; não editar.
+
+CANVAS_DIMENSIONS, 16,
+PATH_COLOR_ALPHA, 0x38,
+FILL_RULE_NONZERO,
+DISABLE_AA,
+MOVE_TO, 7, 3,
+LINE_TO, 8, 3,
+LINE_TO, 8, 4,
+LINE_TO, 7, 4,
+CLOSE,
+MOVE_TO, 6, 4,
+LINE_TO, 8, 4,
+LINE_TO, 8, 5,
+LINE_TO, 6, 5,
+CLOSE,
+MOVE_TO, 5, 5,
+LINE_TO, 8, 5,
+LINE_TO, 8, 6,
+LINE_TO, 5, 6,
+CLOSE,
+MOVE_TO, 2, 6,
+LINE_TO, 8, 6,
+LINE_TO, 8, 7,
+LINE_TO, 2, 7,
+CLOSE,
+MOVE_TO, 2, 7,
+LINE_TO, 8, 7,
+LINE_TO, 8, 8,
+LINE_TO, 2, 8,
+CLOSE,
+MOVE_TO, 2, 8,
+LINE_TO, 8, 8,
+LINE_TO, 8, 9,
+LINE_TO, 2, 9,
+CLOSE,
+MOVE_TO, 5, 9,
+LINE_TO, 8, 9,
+LINE_TO, 8, 10,
+LINE_TO, 5, 10,
+CLOSE,
+MOVE_TO, 6, 10,
+LINE_TO, 8, 10,
+LINE_TO, 8, 11,
+LINE_TO, 6, 11,
+CLOSE,
+MOVE_TO, 7, 11,
+LINE_TO, 8, 11,
+LINE_TO, 8, 12,
+LINE_TO, 7, 12,
+CLOSE,
+NEW_PATH,
+PATH_COLOR_ALPHA, 0x80,
+FILL_RULE_NONZERO,
+DISABLE_AA,
+MOVE_TO, 9, 3,
+LINE_TO, 10, 3,
+LINE_TO, 10, 4,
+LINE_TO, 9, 4,
+CLOSE,
+MOVE_TO, 9, 4,
+LINE_TO, 10, 4,
+LINE_TO, 10, 5,
+LINE_TO, 9, 5,
+CLOSE,
+MOVE_TO, 9, 5,
+LINE_TO, 10, 5,
+LINE_TO, 10, 6,
+LINE_TO, 9, 6,
+CLOSE,
+MOVE_TO, 9, 6,
+LINE_TO, 10, 6,
+LINE_TO, 10, 7,
+LINE_TO, 9, 7,
+CLOSE,
+MOVE_TO, 9, 7,
+LINE_TO, 10, 7,
+LINE_TO, 10, 8,
+LINE_TO, 9, 8,
+CLOSE,
+MOVE_TO, 9, 8,
+LINE_TO, 10, 8,
+LINE_TO, 10, 9,
+LINE_TO, 9, 9,
+CLOSE,
+MOVE_TO, 9, 9,
+LINE_TO, 10, 9,
+LINE_TO, 10, 10,
+LINE_TO, 9, 10,
+CLOSE,
+MOVE_TO, 2, 10,
+LINE_TO, 5, 10,
+LINE_TO, 5, 11,
+LINE_TO, 2, 11,
+CLOSE,
+MOVE_TO, 9, 10,
+LINE_TO, 10, 10,
+LINE_TO, 10, 11,
+LINE_TO, 9, 11,
+CLOSE,
+MOVE_TO, 5, 11,
+LINE_TO, 6, 11,
+LINE_TO, 6, 12,
+LINE_TO, 5, 12,
+CLOSE,
+MOVE_TO, 9, 11,
+LINE_TO, 10, 11,
+LINE_TO, 10, 12,
+LINE_TO, 9, 12,
+CLOSE,
+MOVE_TO, 6, 12,
+LINE_TO, 7, 12,
+LINE_TO, 7, 13,
+LINE_TO, 6, 13,
+CLOSE,
+MOVE_TO, 7, 13,
+LINE_TO, 8, 13,
+LINE_TO, 8, 14,
+LINE_TO, 7, 14,
+CLOSE,
+NEW_PATH,
+FILL_RULE_NONZERO,
+DISABLE_AA,
+MOVE_TO, 8, 1,
+LINE_TO, 9, 1,
+LINE_TO, 9, 2,
+LINE_TO, 8, 2,
+CLOSE,
+MOVE_TO, 7, 2,
+LINE_TO, 9, 2,
+LINE_TO, 9, 3,
+LINE_TO, 7, 3,
+CLOSE,
+MOVE_TO, 6, 3,
+LINE_TO, 7, 3,
+LINE_TO, 7, 4,
+LINE_TO, 6, 4,
+CLOSE,
+MOVE_TO, 8, 3,
+LINE_TO, 9, 3,
+LINE_TO, 9, 4,
+LINE_TO, 8, 4,
+CLOSE,
+MOVE_TO, 12, 3,
+LINE_TO, 13, 3,
+LINE_TO, 13, 4,
+LINE_TO, 12, 4,
+CLOSE,
+MOVE_TO, 5, 4,
+LINE_TO, 6, 4,
+LINE_TO, 6, 5,
+LINE_TO, 5, 5,
+CLOSE,
+MOVE_TO, 8, 4,
+LINE_TO, 9, 4,
+LINE_TO, 9, 5,
+LINE_TO, 8, 5,
+CLOSE,
+MOVE_TO, 13, 4,
+LINE_TO, 14, 4,
+LINE_TO, 14, 5,
+LINE_TO, 13, 5,
+CLOSE,
+MOVE_TO, 1, 5,
+LINE_TO, 5, 5,
+LINE_TO, 5, 6,
+LINE_TO, 1, 6,
+CLOSE,
+MOVE_TO, 8, 5,
+LINE_TO, 9, 5,
+LINE_TO, 9, 6,
+LINE_TO, 8, 6,
+CLOSE,
+MOVE_TO, 10, 5,
+LINE_TO, 11, 5,
+LINE_TO, 11, 6,
+LINE_TO, 10, 6,
+CLOSE,
+MOVE_TO, 13, 5,
+LINE_TO, 14, 5,
+LINE_TO, 14, 6,
+LINE_TO, 13, 6,
+CLOSE,
+MOVE_TO, 1, 6,
+LINE_TO, 2, 6,
+LINE_TO, 2, 7,
+LINE_TO, 1, 7,
+CLOSE,
+MOVE_TO, 8, 6,
+LINE_TO, 9, 6,
+LINE_TO, 9, 7,
+LINE_TO, 8, 7,
+CLOSE,
+MOVE_TO, 11, 6,
+LINE_TO, 12, 6,
+LINE_TO, 12, 7,
+LINE_TO, 11, 7,
+CLOSE,
+MOVE_TO, 13, 6,
+LINE_TO, 14, 6,
+LINE_TO, 14, 7,
+LINE_TO, 13, 7,
+CLOSE,
+MOVE_TO, 1, 7,
+LINE_TO, 2, 7,
+LINE_TO, 2, 8,
+LINE_TO, 1, 8,
+CLOSE,
+MOVE_TO, 8, 7,
+LINE_TO, 9, 7,
+LINE_TO, 9, 8,
+LINE_TO, 8, 8,
+CLOSE,
+MOVE_TO, 11, 7,
+LINE_TO, 12, 7,
+LINE_TO, 12, 8,
+LINE_TO, 11, 8,
+CLOSE,
+MOVE_TO, 13, 7,
+LINE_TO, 14, 7,
+LINE_TO, 14, 8,
+LINE_TO, 13, 8,
+CLOSE,
+MOVE_TO, 1, 8,
+LINE_TO, 2, 8,
+LINE_TO, 2, 9,
+LINE_TO, 1, 9,
+CLOSE,
+MOVE_TO, 8, 8,
+LINE_TO, 9, 8,
+LINE_TO, 9, 9,
+LINE_TO, 8, 9,
+CLOSE,
+MOVE_TO, 11, 8,
+LINE_TO, 12, 8,
+LINE_TO, 12, 9,
+LINE_TO, 11, 9,
+CLOSE,
+MOVE_TO, 13, 8,
+LINE_TO, 14, 8,
+LINE_TO, 14, 9,
+LINE_TO, 13, 9,
+CLOSE,
+MOVE_TO, 1, 9,
+LINE_TO, 5, 9,
+LINE_TO, 5, 10,
+LINE_TO, 1, 10,
+CLOSE,
+MOVE_TO, 8, 9,
+LINE_TO, 9, 9,
+LINE_TO, 9, 10,
+LINE_TO, 8, 10,
+CLOSE,
+MOVE_TO, 10, 9,
+LINE_TO, 11, 9,
+LINE_TO, 11, 10,
+LINE_TO, 10, 10,
+CLOSE,
+MOVE_TO, 13, 9,
+LINE_TO, 14, 9,
+LINE_TO, 14, 10,
+LINE_TO, 13, 10,
+CLOSE,
+MOVE_TO, 5, 10,
+LINE_TO, 6, 10,
+LINE_TO, 6, 11,
+LINE_TO, 5, 11,
+CLOSE,
+MOVE_TO, 8, 10,
+LINE_TO, 9, 10,
+LINE_TO, 9, 11,
+LINE_TO, 8, 11,
+CLOSE,
+MOVE_TO, 13, 10,
+LINE_TO, 14, 10,
+LINE_TO, 14, 11,
+LINE_TO, 13, 11,
+CLOSE,
+MOVE_TO, 6, 11,
+LINE_TO, 7, 11,
+LINE_TO, 7, 12,
+LINE_TO, 6, 12,
+CLOSE,
+MOVE_TO, 8, 11,
+LINE_TO, 9, 11,
+LINE_TO, 9, 12,
+LINE_TO, 8, 12,
+CLOSE,
+MOVE_TO, 12, 11,
+LINE_TO, 13, 11,
+LINE_TO, 13, 12,
+LINE_TO, 12, 12,
+CLOSE,
+MOVE_TO, 7, 12,
+LINE_TO, 9, 12,
+LINE_TO, 9, 13,
+LINE_TO, 7, 13,
+CLOSE,
+MOVE_TO, 8, 13,
+LINE_TO, 9, 13,
+LINE_TO, 9, 14,
+LINE_TO, 8, 14,
+CLOSE,
--- /dev/null
+++ b/chrome/browser/ui/aerographene_theme_icons/pixelized_spkoff.icon
@@ -0,0 +1,270 @@
+// Copyright 2026 The aerographene Authors
+// Use of this source code is governed by the GPL-3.0 license.
+// Gerado por theming/design-language/gen_theme_icons.py; não editar.
+
+CANVAS_DIMENSIONS, 16,
+PATH_COLOR_ALPHA, 0x38,
+FILL_RULE_NONZERO,
+DISABLE_AA,
+MOVE_TO, 7, 3,
+LINE_TO, 8, 3,
+LINE_TO, 8, 4,
+LINE_TO, 7, 4,
+CLOSE,
+MOVE_TO, 6, 4,
+LINE_TO, 8, 4,
+LINE_TO, 8, 5,
+LINE_TO, 6, 5,
+CLOSE,
+MOVE_TO, 5, 5,
+LINE_TO, 8, 5,
+LINE_TO, 8, 6,
+LINE_TO, 5, 6,
+CLOSE,
+MOVE_TO, 2, 6,
+LINE_TO, 8, 6,
+LINE_TO, 8, 7,
+LINE_TO, 2, 7,
+CLOSE,
+MOVE_TO, 2, 7,
+LINE_TO, 8, 7,
+LINE_TO, 8, 8,
+LINE_TO, 2, 8,
+CLOSE,
+MOVE_TO, 2, 8,
+LINE_TO, 8, 8,
+LINE_TO, 8, 9,
+LINE_TO, 2, 9,
+CLOSE,
+MOVE_TO, 5, 9,
+LINE_TO, 8, 9,
+LINE_TO, 8, 10,
+LINE_TO, 5, 10,
+CLOSE,
+MOVE_TO, 6, 10,
+LINE_TO, 8, 10,
+LINE_TO, 8, 11,
+LINE_TO, 6, 11,
+CLOSE,
+MOVE_TO, 7, 11,
+LINE_TO, 8, 11,
+LINE_TO, 8, 12,
+LINE_TO, 7, 12,
+CLOSE,
+NEW_PATH,
+PATH_COLOR_ALPHA, 0x80,
+FILL_RULE_NONZERO,
+DISABLE_AA,
+MOVE_TO, 9, 3,
+LINE_TO, 10, 3,
+LINE_TO, 10, 4,
+LINE_TO, 9, 4,
+CLOSE,
+MOVE_TO, 9, 4,
+LINE_TO, 10, 4,
+LINE_TO, 10, 5,
+LINE_TO, 9, 5,
+CLOSE,
+MOVE_TO, 9, 5,
+LINE_TO, 10, 5,
+LINE_TO, 10, 6,
+LINE_TO, 9, 6,
+CLOSE,
+MOVE_TO, 9, 6,
+LINE_TO, 10, 6,
+LINE_TO, 10, 7,
+LINE_TO, 9, 7,
+CLOSE,
+MOVE_TO, 9, 7,
+LINE_TO, 10, 7,
+LINE_TO, 10, 8,
+LINE_TO, 9, 8,
+CLOSE,
+MOVE_TO, 9, 8,
+LINE_TO, 10, 8,
+LINE_TO, 10, 9,
+LINE_TO, 9, 9,
+CLOSE,
+MOVE_TO, 9, 9,
+LINE_TO, 10, 9,
+LINE_TO, 10, 10,
+LINE_TO, 9, 10,
+CLOSE,
+MOVE_TO, 2, 10,
+LINE_TO, 5, 10,
+LINE_TO, 5, 11,
+LINE_TO, 2, 11,
+CLOSE,
+MOVE_TO, 9, 10,
+LINE_TO, 10, 10,
+LINE_TO, 10, 11,
+LINE_TO, 9, 11,
+CLOSE,
+MOVE_TO, 5, 11,
+LINE_TO, 6, 11,
+LINE_TO, 6, 12,
+LINE_TO, 5, 12,
+CLOSE,
+MOVE_TO, 9, 11,
+LINE_TO, 10, 11,
+LINE_TO, 10, 12,
+LINE_TO, 9, 12,
+CLOSE,
+MOVE_TO, 6, 12,
+LINE_TO, 7, 12,
+LINE_TO, 7, 13,
+LINE_TO, 6, 13,
+CLOSE,
+MOVE_TO, 7, 13,
+LINE_TO, 8, 13,
+LINE_TO, 8, 14,
+LINE_TO, 7, 14,
+CLOSE,
+NEW_PATH,
+FILL_RULE_NONZERO,
+DISABLE_AA,
+MOVE_TO, 8, 1,
+LINE_TO, 9, 1,
+LINE_TO, 9, 2,
+LINE_TO, 8, 2,
+CLOSE,
+MOVE_TO, 7, 2,
+LINE_TO, 9, 2,
+LINE_TO, 9, 3,
+LINE_TO, 7, 3,
+CLOSE,
+MOVE_TO, 6, 3,
+LINE_TO, 7, 3,
+LINE_TO, 7, 4,
+LINE_TO, 6, 4,
+CLOSE,
+MOVE_TO, 8, 3,
+LINE_TO, 9, 3,
+LINE_TO, 9, 4,
+LINE_TO, 8, 4,
+CLOSE,
+MOVE_TO, 5, 4,
+LINE_TO, 6, 4,
+LINE_TO, 6, 5,
+LINE_TO, 5, 5,
+CLOSE,
+MOVE_TO, 8, 4,
+LINE_TO, 9, 4,
+LINE_TO, 9, 5,
+LINE_TO, 8, 5,
+CLOSE,
+MOVE_TO, 1, 5,
+LINE_TO, 5, 5,
+LINE_TO, 5, 6,
+LINE_TO, 1, 6,
+CLOSE,
+MOVE_TO, 8, 5,
+LINE_TO, 9, 5,
+LINE_TO, 9, 6,
+LINE_TO, 8, 6,
+CLOSE,
+MOVE_TO, 10, 5,
+LINE_TO, 11, 5,
+LINE_TO, 11, 6,
+LINE_TO, 10, 6,
+CLOSE,
+MOVE_TO, 13, 5,
+LINE_TO, 14, 5,
+LINE_TO, 14, 6,
+LINE_TO, 13, 6,
+CLOSE,
+MOVE_TO, 1, 6,
+LINE_TO, 2, 6,
+LINE_TO, 2, 7,
+LINE_TO, 1, 7,
+CLOSE,
+MOVE_TO, 8, 6,
+LINE_TO, 9, 6,
+LINE_TO, 9, 7,
+LINE_TO, 8, 7,
+CLOSE,
+MOVE_TO, 11, 6,
+LINE_TO, 13, 6,
+LINE_TO, 13, 7,
+LINE_TO, 11, 7,
+CLOSE,
+MOVE_TO, 1, 7,
+LINE_TO, 2, 7,
+LINE_TO, 2, 8,
+LINE_TO, 1, 8,
+CLOSE,
+MOVE_TO, 8, 7,
+LINE_TO, 9, 7,
+LINE_TO, 9, 8,
+LINE_TO, 8, 8,
+CLOSE,
+MOVE_TO, 11, 7,
+LINE_TO, 13, 7,
+LINE_TO, 13, 8,
+LINE_TO, 11, 8,
+CLOSE,
+MOVE_TO, 1, 8,
+LINE_TO, 2, 8,
+LINE_TO, 2, 9,
+LINE_TO, 1, 9,
+CLOSE,
+MOVE_TO, 8, 8,
+LINE_TO, 9, 8,
+LINE_TO, 9, 9,
+LINE_TO, 8, 9,
+CLOSE,
+MOVE_TO, 11, 8,
+LINE_TO, 13, 8,
+LINE_TO, 13, 9,
+LINE_TO, 11, 9,
+CLOSE,
+MOVE_TO, 1, 9,
+LINE_TO, 5, 9,
+LINE_TO, 5, 10,
+LINE_TO, 1, 10,
+CLOSE,
+MOVE_TO, 8, 9,
+LINE_TO, 9, 9,
+LINE_TO, 9, 10,
+LINE_TO, 8, 10,
+CLOSE,
+MOVE_TO, 10, 9,
+LINE_TO, 11, 9,
+LINE_TO, 11, 10,
+LINE_TO, 10, 10,
+CLOSE,
+MOVE_TO, 13, 9,
+LINE_TO, 14, 9,
+LINE_TO, 14, 10,
+LINE_TO, 13, 10,
+CLOSE,
+MOVE_TO, 5, 10,
+LINE_TO, 6, 10,
+LINE_TO, 6, 11,
+LINE_TO, 5, 11,
+CLOSE,
+MOVE_TO, 8, 10,
+LINE_TO, 9, 10,
+LINE_TO, 9, 11,
+LINE_TO, 8, 11,
+CLOSE,
+MOVE_TO, 6, 11,
+LINE_TO, 7, 11,
+LINE_TO, 7, 12,
+LINE_TO, 6, 12,
+CLOSE,
+MOVE_TO, 8, 11,
+LINE_TO, 9, 11,
+LINE_TO, 9, 12,
+LINE_TO, 8, 12,
+CLOSE,
+MOVE_TO, 7, 12,
+LINE_TO, 9, 12,
+LINE_TO, 9, 13,
+LINE_TO, 7, 13,
+CLOSE,
+MOVE_TO, 8, 13,
+LINE_TO, 9, 13,
+LINE_TO, 9, 14,
+LINE_TO, 8, 14,
+CLOSE,
--- /dev/null
+++ b/chrome/browser/ui/aerographene_theme_icons/pixelized_split.icon
@@ -0,0 +1,305 @@
+// Copyright 2026 The aerographene Authors
+// Use of this source code is governed by the GPL-3.0 license.
+// Gerado por theming/design-language/gen_theme_icons.py; não editar.
+
+CANVAS_DIMENSIONS, 16,
+PATH_COLOR_ALPHA, 0x38,
+FILL_RULE_NONZERO,
+DISABLE_AA,
+MOVE_TO, 2, 2,
+LINE_TO, 7, 2,
+LINE_TO, 7, 3,
+LINE_TO, 2, 3,
+CLOSE,
+MOVE_TO, 2, 3,
+LINE_TO, 7, 3,
+LINE_TO, 7, 4,
+LINE_TO, 2, 4,
+CLOSE,
+MOVE_TO, 2, 4,
+LINE_TO, 7, 4,
+LINE_TO, 7, 5,
+LINE_TO, 2, 5,
+CLOSE,
+MOVE_TO, 2, 5,
+LINE_TO, 7, 5,
+LINE_TO, 7, 6,
+LINE_TO, 2, 6,
+CLOSE,
+MOVE_TO, 2, 6,
+LINE_TO, 7, 6,
+LINE_TO, 7, 7,
+LINE_TO, 2, 7,
+CLOSE,
+MOVE_TO, 2, 7,
+LINE_TO, 7, 7,
+LINE_TO, 7, 8,
+LINE_TO, 2, 8,
+CLOSE,
+MOVE_TO, 2, 8,
+LINE_TO, 7, 8,
+LINE_TO, 7, 9,
+LINE_TO, 2, 9,
+CLOSE,
+MOVE_TO, 2, 9,
+LINE_TO, 7, 9,
+LINE_TO, 7, 10,
+LINE_TO, 2, 10,
+CLOSE,
+MOVE_TO, 2, 10,
+LINE_TO, 7, 10,
+LINE_TO, 7, 11,
+LINE_TO, 2, 11,
+CLOSE,
+MOVE_TO, 2, 11,
+LINE_TO, 7, 11,
+LINE_TO, 7, 12,
+LINE_TO, 2, 12,
+CLOSE,
+MOVE_TO, 2, 12,
+LINE_TO, 7, 12,
+LINE_TO, 7, 13,
+LINE_TO, 2, 13,
+CLOSE,
+NEW_PATH,
+PATH_COLOR_ALPHA, 0x80,
+FILL_RULE_NONZERO,
+DISABLE_AA,
+MOVE_TO, 8, 2,
+LINE_TO, 9, 2,
+LINE_TO, 9, 3,
+LINE_TO, 8, 3,
+CLOSE,
+MOVE_TO, 8, 3,
+LINE_TO, 9, 3,
+LINE_TO, 9, 4,
+LINE_TO, 8, 4,
+CLOSE,
+MOVE_TO, 8, 4,
+LINE_TO, 9, 4,
+LINE_TO, 9, 5,
+LINE_TO, 8, 5,
+CLOSE,
+MOVE_TO, 8, 5,
+LINE_TO, 9, 5,
+LINE_TO, 9, 6,
+LINE_TO, 8, 6,
+CLOSE,
+MOVE_TO, 8, 6,
+LINE_TO, 9, 6,
+LINE_TO, 9, 7,
+LINE_TO, 8, 7,
+CLOSE,
+MOVE_TO, 8, 7,
+LINE_TO, 9, 7,
+LINE_TO, 9, 8,
+LINE_TO, 8, 8,
+CLOSE,
+MOVE_TO, 8, 8,
+LINE_TO, 9, 8,
+LINE_TO, 9, 9,
+LINE_TO, 8, 9,
+CLOSE,
+MOVE_TO, 8, 9,
+LINE_TO, 9, 9,
+LINE_TO, 9, 10,
+LINE_TO, 8, 10,
+CLOSE,
+MOVE_TO, 8, 10,
+LINE_TO, 9, 10,
+LINE_TO, 9, 11,
+LINE_TO, 8, 11,
+CLOSE,
+MOVE_TO, 8, 11,
+LINE_TO, 9, 11,
+LINE_TO, 9, 12,
+LINE_TO, 8, 12,
+CLOSE,
+MOVE_TO, 8, 12,
+LINE_TO, 9, 12,
+LINE_TO, 9, 13,
+LINE_TO, 8, 13,
+CLOSE,
+MOVE_TO, 2, 14,
+LINE_TO, 7, 14,
+LINE_TO, 7, 15,
+LINE_TO, 2, 15,
+CLOSE,
+NEW_PATH,
+FILL_RULE_NONZERO,
+DISABLE_AA,
+MOVE_TO, 1, 1,
+LINE_TO, 14, 1,
+LINE_TO, 14, 2,
+LINE_TO, 1, 2,
+CLOSE,
+MOVE_TO, 1, 2,
+LINE_TO, 2, 2,
+LINE_TO, 2, 3,
+LINE_TO, 1, 3,
+CLOSE,
+MOVE_TO, 7, 2,
+LINE_TO, 8, 2,
+LINE_TO, 8, 3,
+LINE_TO, 7, 3,
+CLOSE,
+MOVE_TO, 13, 2,
+LINE_TO, 14, 2,
+LINE_TO, 14, 3,
+LINE_TO, 13, 3,
+CLOSE,
+MOVE_TO, 1, 3,
+LINE_TO, 2, 3,
+LINE_TO, 2, 4,
+LINE_TO, 1, 4,
+CLOSE,
+MOVE_TO, 7, 3,
+LINE_TO, 8, 3,
+LINE_TO, 8, 4,
+LINE_TO, 7, 4,
+CLOSE,
+MOVE_TO, 13, 3,
+LINE_TO, 14, 3,
+LINE_TO, 14, 4,
+LINE_TO, 13, 4,
+CLOSE,
+MOVE_TO, 1, 4,
+LINE_TO, 2, 4,
+LINE_TO, 2, 5,
+LINE_TO, 1, 5,
+CLOSE,
+MOVE_TO, 7, 4,
+LINE_TO, 8, 4,
+LINE_TO, 8, 5,
+LINE_TO, 7, 5,
+CLOSE,
+MOVE_TO, 13, 4,
+LINE_TO, 14, 4,
+LINE_TO, 14, 5,
+LINE_TO, 13, 5,
+CLOSE,
+MOVE_TO, 1, 5,
+LINE_TO, 2, 5,
+LINE_TO, 2, 6,
+LINE_TO, 1, 6,
+CLOSE,
+MOVE_TO, 7, 5,
+LINE_TO, 8, 5,
+LINE_TO, 8, 6,
+LINE_TO, 7, 6,
+CLOSE,
+MOVE_TO, 13, 5,
+LINE_TO, 14, 5,
+LINE_TO, 14, 6,
+LINE_TO, 13, 6,
+CLOSE,
+MOVE_TO, 1, 6,
+LINE_TO, 2, 6,
+LINE_TO, 2, 7,
+LINE_TO, 1, 7,
+CLOSE,
+MOVE_TO, 7, 6,
+LINE_TO, 8, 6,
+LINE_TO, 8, 7,
+LINE_TO, 7, 7,
+CLOSE,
+MOVE_TO, 13, 6,
+LINE_TO, 14, 6,
+LINE_TO, 14, 7,
+LINE_TO, 13, 7,
+CLOSE,
+MOVE_TO, 1, 7,
+LINE_TO, 2, 7,
+LINE_TO, 2, 8,
+LINE_TO, 1, 8,
+CLOSE,
+MOVE_TO, 7, 7,
+LINE_TO, 8, 7,
+LINE_TO, 8, 8,
+LINE_TO, 7, 8,
+CLOSE,
+MOVE_TO, 13, 7,
+LINE_TO, 14, 7,
+LINE_TO, 14, 8,
+LINE_TO, 13, 8,
+CLOSE,
+MOVE_TO, 1, 8,
+LINE_TO, 2, 8,
+LINE_TO, 2, 9,
+LINE_TO, 1, 9,
+CLOSE,
+MOVE_TO, 7, 8,
+LINE_TO, 8, 8,
+LINE_TO, 8, 9,
+LINE_TO, 7, 9,
+CLOSE,
+MOVE_TO, 13, 8,
+LINE_TO, 14, 8,
+LINE_TO, 14, 9,
+LINE_TO, 13, 9,
+CLOSE,
+MOVE_TO, 1, 9,
+LINE_TO, 2, 9,
+LINE_TO, 2, 10,
+LINE_TO, 1, 10,
+CLOSE,
+MOVE_TO, 7, 9,
+LINE_TO, 8, 9,
+LINE_TO, 8, 10,
+LINE_TO, 7, 10,
+CLOSE,
+MOVE_TO, 13, 9,
+LINE_TO, 14, 9,
+LINE_TO, 14, 10,
+LINE_TO, 13, 10,
+CLOSE,
+MOVE_TO, 1, 10,
+LINE_TO, 2, 10,
+LINE_TO, 2, 11,
+LINE_TO, 1, 11,
+CLOSE,
+MOVE_TO, 7, 10,
+LINE_TO, 8, 10,
+LINE_TO, 8, 11,
+LINE_TO, 7, 11,
+CLOSE,
+MOVE_TO, 13, 10,
+LINE_TO, 14, 10,
+LINE_TO, 14, 11,
+LINE_TO, 13, 11,
+CLOSE,
+MOVE_TO, 1, 11,
+LINE_TO, 2, 11,
+LINE_TO, 2, 12,
+LINE_TO, 1, 12,
+CLOSE,
+MOVE_TO, 7, 11,
+LINE_TO, 8, 11,
+LINE_TO, 8, 12,
+LINE_TO, 7, 12,
+CLOSE,
+MOVE_TO, 13, 11,
+LINE_TO, 14, 11,
+LINE_TO, 14, 12,
+LINE_TO, 13, 12,
+CLOSE,
+MOVE_TO, 1, 12,
+LINE_TO, 2, 12,
+LINE_TO, 2, 13,
+LINE_TO, 1, 13,
+CLOSE,
+MOVE_TO, 7, 12,
+LINE_TO, 8, 12,
+LINE_TO, 8, 13,
+LINE_TO, 7, 13,
+CLOSE,
+MOVE_TO, 13, 12,
+LINE_TO, 14, 12,
+LINE_TO, 14, 13,
+LINE_TO, 13, 13,
+CLOSE,
+MOVE_TO, 1, 13,
+LINE_TO, 14, 13,
+LINE_TO, 14, 14,
+LINE_TO, 1, 14,
+CLOSE,
--- /dev/null
+++ b/chrome/browser/ui/aerographene_theme_icons/pixelized_up.icon
@@ -0,0 +1,112 @@
+// Copyright 2026 The aerographene Authors
+// Use of this source code is governed by the GPL-3.0 license.
+// Gerado por theming/design-language/gen_theme_icons.py; não editar.
+
+CANVAS_DIMENSIONS, 16,
+FILL_RULE_NONZERO,
+DISABLE_AA,
+MOVE_TO, 7, 1,
+LINE_TO, 8, 1,
+LINE_TO, 8, 2,
+LINE_TO, 7, 2,
+CLOSE,
+MOVE_TO, 6, 2,
+LINE_TO, 9, 2,
+LINE_TO, 9, 3,
+LINE_TO, 6, 3,
+CLOSE,
+MOVE_TO, 5, 3,
+LINE_TO, 6, 3,
+LINE_TO, 6, 4,
+LINE_TO, 5, 4,
+CLOSE,
+MOVE_TO, 7, 3,
+LINE_TO, 8, 3,
+LINE_TO, 8, 4,
+LINE_TO, 7, 4,
+CLOSE,
+MOVE_TO, 9, 3,
+LINE_TO, 10, 3,
+LINE_TO, 10, 4,
+LINE_TO, 9, 4,
+CLOSE,
+MOVE_TO, 4, 4,
+LINE_TO, 5, 4,
+LINE_TO, 5, 5,
+LINE_TO, 4, 5,
+CLOSE,
+MOVE_TO, 7, 4,
+LINE_TO, 8, 4,
+LINE_TO, 8, 5,
+LINE_TO, 7, 5,
+CLOSE,
+MOVE_TO, 10, 4,
+LINE_TO, 11, 4,
+LINE_TO, 11, 5,
+LINE_TO, 10, 5,
+CLOSE,
+MOVE_TO, 3, 5,
+LINE_TO, 4, 5,
+LINE_TO, 4, 6,
+LINE_TO, 3, 6,
+CLOSE,
+MOVE_TO, 7, 5,
+LINE_TO, 8, 5,
+LINE_TO, 8, 6,
+LINE_TO, 7, 6,
+CLOSE,
+MOVE_TO, 11, 5,
+LINE_TO, 12, 5,
+LINE_TO, 12, 6,
+LINE_TO, 11, 6,
+CLOSE,
+MOVE_TO, 2, 6,
+LINE_TO, 3, 6,
+LINE_TO, 3, 7,
+LINE_TO, 2, 7,
+CLOSE,
+MOVE_TO, 7, 6,
+LINE_TO, 8, 6,
+LINE_TO, 8, 7,
+LINE_TO, 7, 7,
+CLOSE,
+MOVE_TO, 12, 6,
+LINE_TO, 13, 6,
+LINE_TO, 13, 7,
+LINE_TO, 12, 7,
+CLOSE,
+MOVE_TO, 7, 7,
+LINE_TO, 8, 7,
+LINE_TO, 8, 8,
+LINE_TO, 7, 8,
+CLOSE,
+MOVE_TO, 7, 8,
+LINE_TO, 8, 8,
+LINE_TO, 8, 9,
+LINE_TO, 7, 9,
+CLOSE,
+MOVE_TO, 7, 9,
+LINE_TO, 8, 9,
+LINE_TO, 8, 10,
+LINE_TO, 7, 10,
+CLOSE,
+MOVE_TO, 7, 10,
+LINE_TO, 8, 10,
+LINE_TO, 8, 11,
+LINE_TO, 7, 11,
+CLOSE,
+MOVE_TO, 7, 11,
+LINE_TO, 8, 11,
+LINE_TO, 8, 12,
+LINE_TO, 7, 12,
+CLOSE,
+MOVE_TO, 7, 12,
+LINE_TO, 8, 12,
+LINE_TO, 8, 13,
+LINE_TO, 7, 13,
+CLOSE,
+MOVE_TO, 7, 13,
+LINE_TO, 8, 13,
+LINE_TO, 8, 14,
+LINE_TO, 7, 14,
+CLOSE,
--- /dev/null
+++ b/chrome/browser/ui/aerographene_theme_icons/pixelized_win.icon
@@ -0,0 +1,215 @@
+// Copyright 2026 The aerographene Authors
+// Use of this source code is governed by the GPL-3.0 license.
+// Gerado por theming/design-language/gen_theme_icons.py; não editar.
+
+CANVAS_DIMENSIONS, 16,
+PATH_COLOR_ALPHA, 0x38,
+FILL_RULE_NONZERO,
+DISABLE_AA,
+MOVE_TO, 2, 5,
+LINE_TO, 13, 5,
+LINE_TO, 13, 6,
+LINE_TO, 2, 6,
+CLOSE,
+MOVE_TO, 2, 6,
+LINE_TO, 13, 6,
+LINE_TO, 13, 7,
+LINE_TO, 2, 7,
+CLOSE,
+MOVE_TO, 2, 7,
+LINE_TO, 13, 7,
+LINE_TO, 13, 8,
+LINE_TO, 2, 8,
+CLOSE,
+MOVE_TO, 2, 8,
+LINE_TO, 13, 8,
+LINE_TO, 13, 9,
+LINE_TO, 2, 9,
+CLOSE,
+MOVE_TO, 2, 9,
+LINE_TO, 13, 9,
+LINE_TO, 13, 10,
+LINE_TO, 2, 10,
+CLOSE,
+MOVE_TO, 2, 10,
+LINE_TO, 13, 10,
+LINE_TO, 13, 11,
+LINE_TO, 2, 11,
+CLOSE,
+MOVE_TO, 2, 11,
+LINE_TO, 13, 11,
+LINE_TO, 13, 12,
+LINE_TO, 2, 12,
+CLOSE,
+MOVE_TO, 2, 12,
+LINE_TO, 13, 12,
+LINE_TO, 13, 13,
+LINE_TO, 2, 13,
+CLOSE,
+NEW_PATH,
+PATH_COLOR_ALPHA, 0x80,
+FILL_RULE_NONZERO,
+DISABLE_AA,
+MOVE_TO, 14, 5,
+LINE_TO, 15, 5,
+LINE_TO, 15, 6,
+LINE_TO, 14, 6,
+CLOSE,
+MOVE_TO, 14, 6,
+LINE_TO, 15, 6,
+LINE_TO, 15, 7,
+LINE_TO, 14, 7,
+CLOSE,
+MOVE_TO, 14, 7,
+LINE_TO, 15, 7,
+LINE_TO, 15, 8,
+LINE_TO, 14, 8,
+CLOSE,
+MOVE_TO, 14, 8,
+LINE_TO, 15, 8,
+LINE_TO, 15, 9,
+LINE_TO, 14, 9,
+CLOSE,
+MOVE_TO, 14, 9,
+LINE_TO, 15, 9,
+LINE_TO, 15, 10,
+LINE_TO, 14, 10,
+CLOSE,
+MOVE_TO, 14, 10,
+LINE_TO, 15, 10,
+LINE_TO, 15, 11,
+LINE_TO, 14, 11,
+CLOSE,
+MOVE_TO, 14, 11,
+LINE_TO, 15, 11,
+LINE_TO, 15, 12,
+LINE_TO, 14, 12,
+CLOSE,
+MOVE_TO, 14, 12,
+LINE_TO, 15, 12,
+LINE_TO, 15, 13,
+LINE_TO, 14, 13,
+CLOSE,
+MOVE_TO, 2, 14,
+LINE_TO, 13, 14,
+LINE_TO, 13, 15,
+LINE_TO, 2, 15,
+CLOSE,
+NEW_PATH,
+FILL_RULE_NONZERO,
+DISABLE_AA,
+MOVE_TO, 1, 1,
+LINE_TO, 14, 1,
+LINE_TO, 14, 2,
+LINE_TO, 1, 2,
+CLOSE,
+MOVE_TO, 1, 2,
+LINE_TO, 2, 2,
+LINE_TO, 2, 3,
+LINE_TO, 1, 3,
+CLOSE,
+MOVE_TO, 13, 2,
+LINE_TO, 14, 2,
+LINE_TO, 14, 3,
+LINE_TO, 13, 3,
+CLOSE,
+MOVE_TO, 1, 3,
+LINE_TO, 2, 3,
+LINE_TO, 2, 4,
+LINE_TO, 1, 4,
+CLOSE,
+MOVE_TO, 13, 3,
+LINE_TO, 14, 3,
+LINE_TO, 14, 4,
+LINE_TO, 13, 4,
+CLOSE,
+MOVE_TO, 1, 4,
+LINE_TO, 14, 4,
+LINE_TO, 14, 5,
+LINE_TO, 1, 5,
+CLOSE,
+MOVE_TO, 1, 5,
+LINE_TO, 2, 5,
+LINE_TO, 2, 6,
+LINE_TO, 1, 6,
+CLOSE,
+MOVE_TO, 13, 5,
+LINE_TO, 14, 5,
+LINE_TO, 14, 6,
+LINE_TO, 13, 6,
+CLOSE,
+MOVE_TO, 1, 6,
+LINE_TO, 2, 6,
+LINE_TO, 2, 7,
+LINE_TO, 1, 7,
+CLOSE,
+MOVE_TO, 13, 6,
+LINE_TO, 14, 6,
+LINE_TO, 14, 7,
+LINE_TO, 13, 7,
+CLOSE,
+MOVE_TO, 1, 7,
+LINE_TO, 2, 7,
+LINE_TO, 2, 8,
+LINE_TO, 1, 8,
+CLOSE,
+MOVE_TO, 13, 7,
+LINE_TO, 14, 7,
+LINE_TO, 14, 8,
+LINE_TO, 13, 8,
+CLOSE,
+MOVE_TO, 1, 8,
+LINE_TO, 2, 8,
+LINE_TO, 2, 9,
+LINE_TO, 1, 9,
+CLOSE,
+MOVE_TO, 13, 8,
+LINE_TO, 14, 8,
+LINE_TO, 14, 9,
+LINE_TO, 13, 9,
+CLOSE,
+MOVE_TO, 1, 9,
+LINE_TO, 2, 9,
+LINE_TO, 2, 10,
+LINE_TO, 1, 10,
+CLOSE,
+MOVE_TO, 13, 9,
+LINE_TO, 14, 9,
+LINE_TO, 14, 10,
+LINE_TO, 13, 10,
+CLOSE,
+MOVE_TO, 1, 10,
+LINE_TO, 2, 10,
+LINE_TO, 2, 11,
+LINE_TO, 1, 11,
+CLOSE,
+MOVE_TO, 13, 10,
+LINE_TO, 14, 10,
+LINE_TO, 14, 11,
+LINE_TO, 13, 11,
+CLOSE,
+MOVE_TO, 1, 11,
+LINE_TO, 2, 11,
+LINE_TO, 2, 12,
+LINE_TO, 1, 12,
+CLOSE,
+MOVE_TO, 13, 11,
+LINE_TO, 14, 11,
+LINE_TO, 14, 12,
+LINE_TO, 13, 12,
+CLOSE,
+MOVE_TO, 1, 12,
+LINE_TO, 2, 12,
+LINE_TO, 2, 13,
+LINE_TO, 1, 13,
+CLOSE,
+MOVE_TO, 13, 12,
+LINE_TO, 14, 12,
+LINE_TO, 14, 13,
+LINE_TO, 13, 13,
+CLOSE,
+MOVE_TO, 1, 13,
+LINE_TO, 14, 13,
+LINE_TO, 14, 14,
+LINE_TO, 1, 14,
+CLOSE,
